<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Human Practices | iGEM Aachen</title><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../css/content.css" rel="stylesheet"></head><body data-spy="scroll" data-target="#contentnav" data-offset="100"><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="/"><img src="../assets/img/teamlogo_2021.svg"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="/Description">Description</a><a class="dropdown-item" href="/Implementation">Implementation</a><a class="dropdown-item" href="/Engineering">Engineering</a><a class="dropdown-item" href="/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="/Contribution">Contribution</a><a class="dropdown-item" href="/Outlook">Outlook</a><a class="dropdown-item" href="/Memes">Memes</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarWetlabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Wetlab</a><div class="dropdown-menu" aria-labelledby="navbarWetlabDropdown"><a class="dropdown-item" href="/Experiments">Experiments</a><a class="dropdown-item" href="/Notebook">Notebook</a><a class="dropdown-item" href="/Protocols">Protocols</a><a class="dropdown-item" href="/Results">Results</a><a class="dropdown-item" href="/Safety">Safety</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarDrylabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Drylab</a><div class="dropdown-menu" aria-labelledby="navbarDrylabDropdown"><a class="dropdown-item" href="/Hardware">Hardware</a><a class="dropdown-item" href="/Model">Modeling</a><a class="dropdown-item" href="/Software">Software</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="/Human_Practices">Human Practices</a><a class="dropdown-item" href="/Communication">Communication</a><a class="dropdown-item" href="/Education">Education</a><a class="dropdown-item" href="/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="/Team">Team</a><a class="dropdown-item" href="/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="../assets/img/headers/Human Practices.jpg"><img class="header-text" src="../assets/img/headers/header_texts/Human Practices.svg"><img class="header-effect" src="../assets/img/headers/header_effect.svg"></header><nav id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Conversation at the city archive in Aachen</h1><p>In order to be able to adapt our ideas to the later application, we got into contact with a potential user of our DNA data storage. We visited the city archive in Aachen to discuss long-term storage. Archive manager Dr. René Rohrkamp revealed to us that the primary task of an archive is the disposal of information. Only 1-2% of the local information is stored in a city archive. The goal of Dr. Rohrkamp and the employees is to store data in such a way that even after a long time, it is possible to recognize what everyday life in Aachen used to be like. The city archive is currently in the process of digitalizing many documents and storing them in a long-term repository (server archive) in Cologne. However, this includes many problems: First, extremely high costs are involved. Second, the current data storage devices are magnetic tapes with data decay and lack of robustness. These storage units must be regularly renewed. Third, for security, the electronic signature of the memories must be changed every few years as a seal of quality and a warning signal in case of hacker attacks. During our discussion, we also addressed the topic of data loss. Today much is clarified via email and not signed, stamped, and stored, which is why data is lost and a black hole of information is created. Furthermore, we wanted to know if the value of a document is diminished by digital storage. Digitalization is desirable, as this makes the documents available to the general public on the Internet. In conclusion, the city archive is looking for cheaper, uniform, and safe data storage.</p><p>How we integrated this information into our project can be observed in <a href="TODO">integrated human practices</a>. Based on the proposals and expectations of the archive manager and employees of the city archive in Aachen we developed a guideline of information storage.</p><div class="image"><img src="../assets/img/discussion_city_archive.jpg" alt="Conversation at the city archive in Aachen"><p>Figure 1: Conversation at the city archive in Aachen</p></div><h1>Meeting with Steven Benner</h1><p>Steven Benner, PhD is one of the pioneers in the field of synthetic biology and an expert in artificial genetic systems. We had the honor to present our topic to him and discuss the integration of unnatural base pairs into our DNA.</p><p>Mr. Benner and his colleagues have developed a six-letter Artificially Expanded Genetic Information System (AEGIS), which includes two nonstandard nucleotides in addition to the four standard nucleotides <a href="#citation1">1</a>. According to Mr. Benner, integrating unnatural nucleotides like Z and P could have two main effects on our storage system: A larger number of available nucleotides opens up new possibilities in coding and increases the storage capacity of the DNA. A second effect is the decrease of secondary structure formation due to less equal sequences. But, this could also have some negative impacts on the analysis of DNA, as unnatural nucleotides are not recognized in nanopore sequencing. Therefore, we have made efforts to find other methods of analysis (more information here)[TODO]. Additionally, unnatural nucleotides are not available on the market, so we have dealt with this issue mainly theoretically (read link to get more information).</p><p>Due to his many years of laboratory experience, Mr. Benner could also provide us with some additional recommendations. Despite the view that the TdT can also accept unnatural nucleotides, this must be observed for our specific kind of TdT and reaction buffer <a href="#citation2">2</a>. Mr. Benner also advised us not to use nucleotides with photolabile groups as their size probably prevents the incorporation by the TdT into ssDNA strands. Based on these considerations, we have rethought our hardware and developed a new system, which is less oriented towards already known systems. Mr. Benner also confirmed that the application of SSBs to avoid secondary structures is worth a try.</p><p>In summary, we had a particularly enriching conversation with some valuable tips and tricks for the implementation of our project.</p><div class="image"><img src="../assets/img/meeting_benner.jpeg" alt="Meeting with Steven Benner"><p>Figure 2: Meeting with Steven Benner</p></div><h1>Meeting with Jost Muth and Lena Julie Freund from the Fraunhofer IME Aachen</h1><p>In order to analyze our samples far more in detail, Sebastian Palluk suggested we to talk to the capillary electrophoresis (CE) experts from the Fraunhofer IME in Aachen. At the IME they use an ABI3730 Genetic Analyzer to measure the length of for example PCR products. For this analysis method, our DNA has to be labeled with a fluorescent dye. Mrs. Freund suggested us the 6FAM dye because of its low costs and good results. Through multiple questions, we concluded that the CE is an optimal analyzing method for our samples because the DNA hasn't to be double-stranded owing to the fact that the samples are denatured beforehand. Furthermore, the size range from 35 nt to 500 nt within 1 nt resolution is the range of our synthesized fragments. After the questions, Mrs. Freund and Mr. Muth offered us to support our project by trying to analyze our samples, help us with the interpretation of the data, and being open to questions in the future. Later, we had some more meetings to work out the analysis of our created DNA samples, which subsequently performed well. Our Modeling team used this data to adapt their model.</p><h1>Meeting with Maximilian Schmidt from FZ Jülich</h1><p>Maximilian Schmidt is a member of the section IBG-4 Bioinformatics in FZ Jülich. We contacted him referring some questions about the Nanopore Sequencing with the MinION from Oxford Nanopore. For our progress in the lab, it is necessary to sequence the DNA strands we created, but there are some difficulties because of our specific method. So, we scheduled some meetings with Maximilian where we presented our project and talked about possible sequencing methods. He offered to join us in the lab and bring his MinION to analyze our samples with his help. It was exciting to learn about the sequence method of the MinION, and it was an excellent opportunity to ask as many questions about it as we wanted. Maximilian was very committed to sharing his experiences with us and was very helpful. At the end of the day, sequencing was successful, and we had a lot of new data for our project progress.</p><div class="image"><img src="../assets/img/Max Schmidt Bild.jpg" alt="Meeting Maximilian Schmidt in the lab in Aachen"><p>Figure 3: Meeting Maximilian Schmidt in the lab in Aachen</p></div><h1>Meeting with Prof. Weinhold</h1><p>Prof. Weinhold accompanied us on our path from the very beginning of our journey in February where we were still figuring out our topic. He is an organic chemist focusing on tools for genomics and proteomics. Weinhold directly was enthusiastic about our project. He pointed out a few challenges we might be facing in course of our work in the wet lab. On our second meeting in March, we discussed the prevention of secondary structures and he advised us to use photolabile protecting groups (read more about it in the outlook). We also discussed the possibility of incorporating unnatural bases and elaborated that Deep Sequencing could be useful in this regard. He then helped us establish contact with Steven Benner. Quite at the end of our journey in September we also turned to Prof. Weinhold when we were struggling with our immobilization system. He advised us to try out the Streptavidin-Biotin magnetic immobilization, which ended up being our preferred system. Weinhold always met us with lots of positivity and enthusiasm and motivated us throughout the competition.</p><h1>Meeting with Dr. Florian Kraft</h1><p>Next-generation sequencing is a modern sequencing method, which enables massively parallel sequencing of numerous genes and has thus revolutionized genetic diagnostics in recent years. Dr. Florian Kraft, post-doc at the Institut for human genetic in the uni hospital in Aachen, deals precisely with New Sequencing techniques, especially Nanopore sequencing, which is our preferrable method to analyze our synthesized sequences. Before our meeting and towards the end of our project we exchanged a lot of E-Mail’s and asked Dr. Kraft many questions about our samples for Nanopore sequencing, which he was always happy to answer. During our meeting in the middle of April, he explained the requirements for Nanopore-sequencing, such as the right primer length for Nanopore-sequencing. He raised the issue that using Homopolymers may be difficult because the nanopore could not detect the length of the polymer. Furthermore, we assessed the problem of adding his double-stranded nanopore adapters to our single-stranded DNA. Eventually, he proposed the opportunity to sequence our samples at the uni hospital at a low cost. We gladly accepted his offer and want to thank Dr. Kraft again for the great opportunity!</p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Yang, Z., Hutter, D., Sheng, P., Sismour, A. M., &amp; Benner, S. A. (2006)</p><cite>Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">34(21), 6095–6101</span></p><a class="in-text" href="https://doi.org/10.1093/nar/gkl633" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Tauraitė, D., Jakubovska, J., Dabužinskaitė, J., Bratchikov, M., &amp; Meškys, R. (2017)</p><cite>Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase</cite><p><span class="journalTitle">Molecules (Basel, Switzerland)</span> <span class="journalInfo">22(4)</span></p><a class="in-text" href="https://doi.org/10.3390/molecules22040672" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="../assets/img/all_sponsors.png"></div></div></footer><script src="../content.bundle.js"></script></body></html>