<html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Human Practices | iGEM Aachen</title><script type="text/x-mathjax-config" async>MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
});</script><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" async></script><script>function resizeSidebar() {document.getElementById("contents").style.height = (document.getElementById("list").scrollHeight + 100 + "px"); }
window.onresize = resizeSidebar;</script><link href="../css/content.css" rel="stylesheet"></head><body data-spy="scroll" data-target="#contentnav" data-offset="150" onload="resizeSidebar()"><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="/"><img src="../assets/img/teamlogo_2021.svg"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="/Description">Description</a><a class="dropdown-item" href="/Implementation">Implementation</a><a class="dropdown-item" href="/Engineering">Engineering Success</a><a class="dropdown-item" href="/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="/Contribution">Contribution</a><a class="dropdown-item" href="/Outlook">Outlook</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarWetlabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Wetlab</a><div class="dropdown-menu" aria-labelledby="navbarWetlabDropdown"><a class="dropdown-item" href="/Experiments">Experiments</a><a class="dropdown-item" href="/Notebook">Notebook</a><a class="dropdown-item" href="/Protocols">Protocols</a><a class="dropdown-item" href="/Results">Results</a><a class="dropdown-item" href="/Safety">Safety</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarDrylabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Drylab</a><div class="dropdown-menu" aria-labelledby="navbarDrylabDropdown"><a class="dropdown-item" href="/Hardware">Hardware</a><a class="dropdown-item" href="/Model">Modeling</a><a class="dropdown-item" href="/Software">Software</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="/Human_Practices">Human Practices</a><a class="dropdown-item" href="/Communication">Science Communication</a><a class="dropdown-item" href="/Collaborations">Collaborations</a><a class="dropdown-item" href="/Memes">Memes</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="/Members">Members</a><a class="dropdown-item" href="/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="../assets/img/headers/Human Practices.jpg"><img class="header-text" src="../assets/img/headers/header_texts/Human Practices.svg"><img class="header-effect" src="../assets/img/headers/header_effect.svg"></header><nav class="bg-image" id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Human Practices</h5><ul></ul></div></div><div class="content col-lg-9"><article><link rel="stylesheet" href="src/assets/lightbox/glightbox.min.css"><h1>Integrated Human Practices</h1><p>Reaching out to the world was a source of inspiration. We integrated new ideas and impulses for wet lab, hardware, modeling, and the usage of data storage systems. That is what made it possible to push our project towards extraordinary success. We were amazed by what we could achieve by broadening our horizons and implementing the gained knowledge. To visualize the progress of integrating new ideas, we put together a timeline.</p><div class="image"><a class="glightbox" href="../assets/img/Human Practices/ihp_timeline.png" data-glightbox="zoomable: true;"><img src="../assets/img/Human Practices/ihp_timeline.png" style="width: 100%"></a></div><h1>Human Practices</h1><p>Connecting to the world is an essential part of science. We take steps beyond the lab and communicate with the world, making science more accessible to every person regardless of their background and age. During our journey, we reached out to different people. We explained the principles of DNA data to potential users, the coworkers of the city archive in Aachen and also exchanged ideas with various experts and professors from different countries around the world.</p><h1>Conversation at the City Archive in Aachen</h1><p>To put our ideas into practice, we reached out to a potential user of our DNA data storage and visited the city archive in Aachen to discuss long-term storage. Archive manager Dr. René Rohrkamp explained to us that the primary task of an archive is the disposal of information. Only 1-2% of the local information is stored in a city archive.</p><p>The goal of Dr. Rohrkamp and the employees is to store data in such a way that even after a long time, it is possible to recognize what everyday life in Aachen used to be. The city archive is currently in the process of digitalizing numerous documents and storing them in a long-term repository (server archive) in Cologne. However, this causes various problems: Firstly, storing information long-term is really expensive. Secondly, the current data storage devices are magnetic tapes, where data decays quickly due to the lack of robustness. These storage units must be regularly renewed. Furthermore, to assure quality and provide safety, the electronic signature of the data must be changed every few years. During our discussion, we also addressed the topic of data loss. Today a lot of information is clarified via email, thus it is neither signed and stamped nor stored, which is why data is lost, and a void is left.</p><p>Digitalization is desirable, as this makes documents available to the general public on the internet. In conclusion, the city archive is looking for cheaper, uniform, and safe data storage. To make the first steps achieving this, we encoded a newspaper article from 1933 in our code (see <a href="https://2021.igem.org/Team:Aachen/Software">Software</a>).</p><div class="carousel slide igem-carousel" id="carousel-1" data-interval="false"><div class="carousel-inner"><div class="carousel-item active"><a class="glightbox" href="../assets/img/Human Practices/city archive.jpg"><img class="d-block w-100" src="../assets/img/Human Practices/city archive.jpg"></a></div><div class="carousel-item"><a class="glightbox" href="../assets/img/Human Practices/Group picture.jpg"><img class="d-block w-100" src="../assets/img/Human Practices/Group picture.jpg"></a></div><div class="carousel-item"><a class="glightbox" href="../assets/img/Human Practices/discussion_city_archive.jpg"><img class="d-block w-100" src="../assets/img/Human Practices/discussion_city_archive.jpg"></a></div></div><a class="carousel-control-prev" href="#carousel-1" role="button" data-slide="prev"><span class="carousel-control-prev-icon" aria-hidden="true"></span><span class="sr-only"></span></a><a class="carousel-control-next" href="#carousel-1" role="button" data-slide="next"><span class="carousel-control-next-icon" aria-hidden="true"></span><span class="sr-only"></span></a></div><div class="image"><img src="../assets/img/Human Practices/Old document.jpg" alt="Old document which is about to be digitilized." style="width: 60%"><p style="font-size:0.8rem; text-align:center"><b>Figure 2</b>: Old document which is about to be digitilized.</p></div><h1>Meeting with George Church</h1><p>George Church, Ph.D. is a Professor at Harvard University, coinventor of nanopore sequencing, and expert for digital data storage using DNA. He was involved in the first genomic sequencing, the Human Genome Project, optimization of CRISPR/Cas9, and many more projects concerning synthetic biology.</p><p>We started into the meeting with a sheet full of questions considering our project implementation. The first plans of our teams were to use a laser to control the TdT reaction in a flow cell. A publication of George Church himself inspired us for this system. He recommended not to use this system due to complexity and high costs and advised to focus on the immobilization of primers.</p><p>Our discussion also focused on the content and meaning of data storage on DNA. When asked what he would store on DNA, Mr. Church replied that the recreation of technology should be stored because humanity got “[…] to a point where nobody really knows how to do things from scratch”. He is convinced that data storage on DNA will be used in the future and revealed that he prefers in vivo storage. Considering the implementation of long-time storage, he suggested storing DNA cold and dry, for example, on the moon. Based on our surprised reaction about his futuristic ideas, he replied: “The future sometimes arrives faster than you think, sometimes slower.”</p><p>We are very grateful that Mr. Church would like to have more meetings with us and that he was able to enrich us with his ideas. We think this was one of the most inspiring meetings we had.</p><div class="image"><img src="../assets/img/Human Practices/meeting_george_church.png" alt=""></div><h1>Meeting with Steven Benner</h1><p>Steven Benner, Ph.D. is one of the pioneers in the field of synthetic biology and an expert in artificial genetic systems. We had the honor to present our topic to him and discuss the integration of unnatural base pairs into our DNA.</p><p>Mr. Benner and his colleagues have developed a six-letter Artificially Expanded Genetic Information System (AEGIS), which includes two nonstandard nucleotides in addition to the four standard nucleotides <a href="#citation1">1</a>. According to Mr. Benner, integrating unnatural nucleotides like Z and P could have two main effects on our storage system: A larger number of available nucleotides opens up new possibilities in coding and increases the storage capacity of the DNA. A second effect is the decrease of secondary structure formation due to less equal sequences.</p><p>The TdT can accept unnatural nucleotides, but it has to be observed if our specific kind of TdT and the reaction buffer are suitable for that purpose, too <a href="#citation2">2</a>. In conclusion, we had an enriching conversation that provided valuable tips and tricks for the implementation of our project.</p><div class="image"><img src="../assets/img/Human Practices/meeting_benner.jpeg" alt=""></div><h1>Meetings with Sebastian Palluk</h1><p>Dr. Sebastian Palluk is co-founder and CTO of Ansa Biotechnologies, a startup developing enzymatic DNA synthesis technologies. He participated in the iGEM competition himself and experienced that synthesizing DNA on his own was challenging. Now he works with a genetically modified TdT for biological applications and aims to improve the synthesis system. Mr. Palluk was probably one of our most important supporters. We met with him three times in total and had frequent email contact during the whole project.</p><p>The first meeting started by presenting our ideas and two approaches to encode data on DNA. Furthermore, we discussed four different ideas to prevent the formation of secondary structures of ssDNA and the affinity of the TdT for 3'OH base modifications. Mr. Palluk commented on our ideas on immobilization. Briefly, he explained streptavidin immobilization on magnetic beads as well as sequencing techniques like nanopore sequencing and capillary electrophoresis to view the length distribution. To conclude our first meeting, Mr. Palluk gave us a great overview of many important tasks we have to tackle in the future and encouraged us to do our best for the project. He believed that our unique interdisciplinary team would create a great project, but implementing everything we planned would be more challenging than we thought.</p><p>In our second meeting with Mr. Palluk at the beginning of May, we presented our possibilities to hinder secondary structures. He recommended starting our first experiments in solution with purification steps to establish our methods. Furthermore, he emphasized that it would be efficient to start working on the immobilization while conducting our first experiments. Mr. Palluk advised us to analyze the length of strands with gel electrophoresis and explained that driving PCR reaction beforehand is not necessary. Afterward, we discussed the possibilities for sequencing our strands, where he explained that Sanger sequencing would only be possible if we performed vector cloning beforehand.</p><p>In our third meeting at the beginning of September, we discussed our ideas for building a hardware and explained the maleimide-cysteine binding system. Additionally, we examined the problem we have faced with our immobilization. With our current system, we would lose lots of maleimides, which is too expensive for us. Mr. Palluk suggested using a peptide tag to bind primers with homologous sequences and sticky-end &quot;overhangs&quot;. The TdT would bind at the overhangs and could be released by heating. Together, we thought of a &quot;plan B&quot; for the immobilization. To create base transitions in solution, we should add primer in excess and let the reaction finish until all nucleotides are incorporated, and add another nucleotide. To put it in a nutshell, Mr. Palluk was an essential expert for us who helped create many solutions to our problem. Nobody knows the TdT better than he does. Thank you again Mr. Palluk, for answering all our questions and emails!</p><div class="image"><img src="../assets/img/Human Practices/2021-09-07_Meeting Palluk.png" alt=""></div><h1>Meetings with Jost Muth and Lena Julie Freund from the Fraunhofer IME Aachen</h1><p>Sebastian Palluk suggested consulting capillary electrophoresis (CE) experts from the Fraunhofer IME in Aachen to analyze our samples in more detail At the Fraunhofer IME they use an ABI3730 Genetic Analyzer to measure the length of nucleic acids. For this analysis method, our DNA had to be labeled with a fluorescent dye. Mrs. Freund suggested using the 6FAM dye because of its low costs and good results. We concluded that the CE is an optimal analysis method for our samples because the DNA does not have to be double-stranded as the samples are denatured before analysis anyway. Furthermore, the size range from 35 nucleotides (nt) to 500 nt within 1 nt resolution is the range of our synthesized fragments. After answering all of our questions, Mrs. Freund and Mr. Muth offered to support our project by trying to analyze our samples, help us with the interpretation of the data, and be open to questions in the future. Later, we had some more meetings to work out the analysis of our DNA samples. Our Modeling team used this data to adapt their model.</p><h1>Meeting with Maximilian Schmidt from FZ Jülich</h1><p>Maximilian Schmidt is a member of the section IBG-4 Bioinformatics in FZ Jülich. We contacted him with questions about the nanopore sequencing with the MinION from Oxford Nanopore. For our progress in wet lab, it is necessary to sequence the strands we created, but we quickly faced difficulties, because our strands were too short and single-stranded. So, we scheduled an online meeting with Maximilian to present our project and talk about possible sequencing methods. He offered to join us in the lab and brought his MinION several days later to analyze our samples. It was exciting to learn about the sequence method of the MinION, and it was an excellent opportunity to ask as many questions about it as we wanted. Maximilian was very committed to sharing his experiences with us and helped us a lot. Therefore, sequencing was successful at the end of the day, and we had a lot of new data for our project progress.</p><div class="image"><img src="../assets/img/Human Practices/Max Schmidt Bild.jpg" alt=""></div><h1>Meetings with Elmar Weinhold</h1><p>Prof. Dr. Elmar Weinhold accompanied us on our path from the very beginning of our journey in February, where we were still figuring out our topic. He is an organic chemist focusing on tools for genomics and proteomics.</p><p>Mr. Weinhold was directly enthusiastic about our project.</p><p>He pointed out a few challenges we might be facing in the course of our work in the lab. On our second meeting in March, we discussed the prevention of secondary structures, and he advised us to use photolabile protecting groups (read more about it in <a href="https://2021.igem.org/Team:Aachen/Outlook#PreventionofSecondaryStructures">Outlook</a>).</p><p>We also discussed the possibility of incorporating unnatural bases and elaborated that Deep Sequencing could be useful in this regard. He then helped us establish contact with Steven Benner. In September, we also turned to Prof. Weinhold when we were struggling with our immobilization system. He advised us to try out the streptavidin-biotin magnetic immobilization, which ended up being our preferred system. Mr. Weinhold always met us with lots of positivity and enthusiasm and motivated us throughout the competition.</p><h1>Meeting with Dörte Rother, Torsten Sehl and Jan-Dirk Spöring</h1><p>In June we met with Prof. Dr. Dörte Rother, Dr. Torsten Sehl and Jan-Dirk Spöring from FZ Jülich from the group of Biocatalysis. Mrs. Rother Rother is an expert in enzyme immobilization and showed us possibilities that suit our purposes.</p><p>Back then, we have not decided whether we should immobilize a biotinylated primer on a streptavidin plate, which George Church inspires, or use our DIP method. Mrs. Rother elaborated every immobilization method with us, showing us advantages and challenges. For the case that we chose the first approach, she warned us that immobilizing the enzyme by dialysis or any other kind of membrane could be disadvantageous because the material fast could block it. For the DIP method, she advised us to immobilize a biotin-labeled primer on a streptavidin-coated magnetic bead and to use a magnetic stick. A few months later, this turned out to be our preferred immobilization method. She also took notice that the efficiency of the enzyme could decrease after the immobilization. In conclusion, we had an informative and pleasant discussion.</p><div class="image"><img src="../assets/img/Human Practices/2021-06-11_Meeting-Rother.png" alt=""></div><h1>Meeting with Martin Zimmermann</h1><p>At the beginning of our project, we had severe problems with our analytics. Gels did not run well, samples were not visible, and many team members had little to no lab experience. Dr. Martin Zimmermann is both a Study Advisor of the Aachen Biology and Biotechnology Department and an expert in many biological fields, including biological analysis methods. During the whole project, we repeatedly exchanged E-Mails, and Mr. Zimmermann examined our issues detailly.</p><p>We met with Mr. Zimmermann at the end of May. He introduced us to denaturing polyacrylamide gels and helped us establish the right analytics for our unconventional samples. Among other topics, we discussed gel runtime, gel thickness, the correct voltage, and staining methods of our gels. To get a better resolution, Mr. Zimmermann advised us to use TBE buffer which has a higher buffer capacity. Furthermore, he suggested silver staining as a sustainable method to stain PAA gels and longer primers to establish better reaction conditions for our enzyme, which we implemented rapidly. At the end of our Meeting, Mr. Zimmermann once more drew attention to polyacrylamide. He advised us to be extra careful when handling the neurotoxic chemical. This was included in our <a href="https://2021.igem.org/Team:Aachen/Safety">Safety Guidelines</a>.</p><h1>Meeting with Florian Kraft</h1><p>Next-generation sequencing is a modern sequencing method, which enables massively parallel sequencing of numerous genes and has thus revolutionized genetic diagnostics in recent years. Dr. Florian Kraft, post-doc at the Institute of Human Genetics at the Uniklinikum in Aachen, deals precisely with new sequencing techniques, especially nanopore sequencing, which is our preferrable method to analyze our synthesized sequences.</p><p>Before our meeting and towards the end of our project, we exchanged a lot of emails and asked Mr. Kraft many questions about our samples for nanopore sequencing, which he was always happy to answer. During our meeting in the middle of April, he explained the requirements for nanopore sequencing, such as the right primer length for nanopore sequencing. He raised the issue that analyzing strands with homopolymers may be difficult because the nanopore could not detect the length of the polymer. Furthermore, we assessed the problem of adding his double-stranded nanopore adapters to our single-stranded DNA. Eventually, he proposed the opportunity to sequence our samples at the University Hospital at a low cost. We gladly accepted his offer and want to thank Mr. Kraft again for the great opportunity!</p><h1>Meeting with Dominik Heider and Anke Becker</h1><p>DNA data storage is a current topic and attracts the attention of scientists from different fields. We believe that it is crucial to meet and exchange ideas with people working on the same topic to drive innovation forward. For this reason, we met with Prof. Dr. Dominik Heider and Prof. Dr. Anke Becker, who are both involved in the project MOSLA (Molecular Storage for Long-term Archiving), at the beginning of June. The project is designated to the research of DNA data storage. Mr. Heider is a bioinformatician working on coding and especially on error correction. Mrs. Becker is working on integrating unnatural bases in the storage code and DNA-based cloud storage in different in vitro and in vivo formats.</p><p>Through the meeting, we were able to present our projects to each other and discuss their research progress. Even though the research group uses phosphoramidite synthesis instead of our enzymatic DNA synthesis system, Mr. Becker could give us helpful advice for our wet lab. She took notice that on a microarray, excess material could block the immobilized primer. Furthermore, she advised us to integrate unnatural base pairs to increase the data density and avoid secondary structures. It was an excellent opportunity to discuss our project with people working on the same subject.</p><h1>Meeting with Jörn Kalinowski and Christian Rückert</h1><p>At the beginning of June, we contacted Prof. Dr. Jörn Kalinowski from the Center for Biotechnology (CeBiTec) in Bielefeld, an expert in genomics and sequencing, hoping to receive support concerning the analysis of samples via nanopore sequencing. Mr. Kalinowksi joined the meeting with his lab coworker Dr. Christian Rückert, an expert in sequencing and an iGEM enthusiast. After listening to a presentation about our project, Mr. Kalinowski and Mr. Rückert explained the principles of nanopore sequencing to us in detail. They noticed that longer strands are preferred, and strands smaller than 200 base pairs are hard to detect. Mr. Rückert then explained all the steps: preparing the strands with a base caller, attaching an adapter and a poly-A tail, and phosphorylating the 5' end. Nanopore sequencing is an important part of our project, and this informative meeting gave us a clearer idea about its usage.</p><h1>Meeting with Alexander Heckel</h1><p>Prof. Dr. Alexander Heckel from Goethe-University in Frankfurt is an expert in photochemistry, and we met with him to discuss the usage of photolabile protecting groups.</p><p>We presented our project and ideas to Mr. Heckel, and he took his time to explain to us the photochemistry of this approach. As soon as those groups are attached to a triphosphate, the bases can not form hydrogen bonds. Thus, we can prevent the formation of secondary structures. The photolabile groups can be detached by a laser as soon as the DNA synthesis reaction is finished.<br>He also offered us to provide those modified triphosphates for our purposes. We are delighted that we had the opportunity of meeting Mr. Heckel!</p><h1>Newsletter for our Sponsors</h1><p>We created newsletters for the important months of May, June, August, and September to inform our sponsors and supporters about our progress. They were written and designed by various members and summarize our progress in the specific month.</p><ul><li><a href="../assets/documents/communication/Newsletter_May_designed.pdf">Newsletter May</a></li><li><a href="../assets/documents/communication/Newsletter_June_designed.pdf">Newsletter June</a></li><li><a href="../assets/documents/communication/Newsletter_August_designed.pdf">Newsletter August</a></li><li><a href="../assets/documents/communication/Newsletter_September.pdf">Newsletter September</a></li></ul></article><script src="src/assets/lightbox/glightbox.min.js"></script><script src="src/assets/lightbox/gallery.js"></script><div class="row justify-content-center"><a class="tourButton" href="https://2021.igem.org/Team:Aachen/Hardware" id="tourButton"><div class="tourButtonDescription"><span class="buttontext">Hardware</span><i class="fas fa-chevron-right" id="arrow"></i></div><div id="tourrocket"></div></a></div><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Yang, Z., Hutter, D., Sheng, P., Sismour, A. M., &amp; Benner, S. A. (2006)</p><cite>Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">34(21), 6095–6101</span></p><a class="in-text" href="https://doi.org/10.1093/nar/gkl633" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Tauraitė, D., Jakubovska, J., Dabužinskaitė, J., Bratchikov, M., &amp; Meškys, R. (2017)</p><cite>Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase</cite><p><span class="journalTitle">Molecules (Basel, Switzerland)</span> <span class="journalInfo">22(4)</span></p><a class="in-text" href="https://doi.org/10.3390/molecules22040672" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="../assets/img/all_sponsors.png"></div></div></footer><script src="../content.bundle.js"></script></body></html>