<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Human Practices | iGEM Aachen</title><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
});</script><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../css/content.css" rel="stylesheet"></head><body data-spy="scroll" data-target="#contentnav" data-offset="100"><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="/"><img src="../assets/img/teamlogo_2021.svg"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="/Description">Description</a><a class="dropdown-item" href="/Implementation">Implementation</a><a class="dropdown-item" href="/Engineering">Engineering</a><a class="dropdown-item" href="/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="/Contribution">Contribution</a><a class="dropdown-item" href="/Memes">Memes</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarWetlabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Wetlab</a><div class="dropdown-menu" aria-labelledby="navbarWetlabDropdown"><a class="dropdown-item" href="/Experiments">Experiments</a><a class="dropdown-item" href="/Notebook">Notebook</a><a class="dropdown-item" href="/Protocols">Protocols</a><a class="dropdown-item" href="/Results">Results</a><a class="dropdown-item" href="/Outlook">Outlook</a><a class="dropdown-item" href="/Safety">Safety</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarDrylabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Drylab</a><div class="dropdown-menu" aria-labelledby="navbarDrylabDropdown"><a class="dropdown-item" href="/Hardware">Hardware</a><a class="dropdown-item" href="/Model">Modeling</a><a class="dropdown-item" href="/Software">Software</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="/Human_Practices">Human Practices</a><a class="dropdown-item" href="/Science_Communication">Science Communication</a><a class="dropdown-item" href="/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="/Members">Members</a><a class="dropdown-item" href="/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="../assets/img/headers/Human Practices.jpg"><img class="header-text" src="../assets/img/headers/header_texts/Human Practices.svg"><img class="header-effect" src="../assets/img/headers/header_effect.svg"></header><nav id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Meeting with George Church</h1><p>George Church, PhD is a Professor at Harvard University, coinventor of nanopore sequencing, and of course expert for digital data storage using DNA. He was involved in the first genomic sequencing, the Human Genome Project, optimization of CRISPR/Cas9, and many more projects concerning synthetic biology.</p><p>We started into the meeting with a sheet full of questions considering our project implementation. The first plans of our teams were to use a laser to control the TdT reaction in a flow cell. This system was inspired by a publication of George Church himself. He recommended not to use this system due to complexity and high costs and advised to focus on the immobilization of primers. Mr. Church was also able to help us with other problems in our project, which can be read about at <a href="https://2021.igem.org/Team:Aachen/Human_Practices">integrated human practices</a>.</p><p>Our discussion also focused on the content and meaning of data storage on DNA. When asked what he would store on DNA, Mr. Church replied that the recreation of technology should be stored because humanity got “[…] to a point where nobody really knows how to do things from scratch”. He is convinced that data storage on DNA will be used in the future and revealed that he prefers in vivo storage. Considering the implementation of long-time storage, he suggested storing DNA cold and dry, e.g. on the moon. Based on our surprised reaction about his futuristic ideas, he replied: “The future sometimes arrives faster than you think, sometimes slower.”</p><p>We are very grateful that Mr. Church would like to have more meetings with us and that he was able to enrich us with his ideas. We think this was one of the most inspiring meetings we had.</p><h1>Conversation at the city archive in Aachen</h1><p>In order to be able to adapt our ideas to the later application, we got into contact with a potential user of our DNA data storage. We visited the city archive in Aachen to discuss long-term storage. Archive manager Dr. René Rohrkamp revealed to us that the primary task of an archive is the disposal of information. Only 1-2% of the local information is stored in a city archive.</p><p>The goal of Dr. Rohrkamp and the employees is to store data in such a way that even after a long time, it is possible to recognize what everyday life in Aachen used to be like. The city archive is currently in the process of digitalizing many documents and storing them in a long-term repository (server archive) in Cologne. However, this includes many problems: First, extremely high costs are involved. Second, the current data storage devices are magnetic tapes with data decay and lack of robustness. These storage units must be regularly renewed. Third, for security, the electronic signature of the memories must be changed every few years as a seal of quality and a warning signal in case of hacker attacks. During our discussion, we also addressed the topic of data loss. Today much is clarified via email and not signed, stamped, and stored, which is why data is lost and a black hole of information is created.</p><p>Furthermore, we wanted to know if the value of a document is diminished by digital storage. Digitalization is desirable, as this makes the documents available to the general public on the Internet. In conclusion, the city archive is looking for cheaper, uniform, and safe data storage. How we integrated this information into our project can be observed at <a href="https://2021.igem.org/Team:Aachen/Human_Practices">integrated human practices</a>.</p><div class="image"><img src="../assets/img/discussion_city_archive.jpg" alt="Conversation at the city archive in Aachen"><p>Figure 1: Conversation at the city archive in Aachen</p></div><h1>Meeting with Steven Benner</h1><p>Steven Benner, PhD is one of the pioneers in the field of synthetic biology and an expert in artificial genetic systems. We had the honor to present our topic to him and discuss the integration of unnatural base pairs into our DNA.</p><p>Mr. Benner and his colleagues have developed a six-letter Artificially Expanded Genetic Information System (AEGIS), which includes two nonstandard nucleotides in addition to the four standard nucleotides <a href="#citation1">1</a>. According to Mr. Benner, integrating unnatural nucleotides like Z and P could have two main effects on our storage system: A larger number of available nucleotides opens up new possibilities in coding and increases the storage capacity of the DNA. A second effect is the decrease of secondary structure formation due to less equal sequences.</p><p>Due to his many years of laboratory experience, Mr. Benner could also provide us with some additional recommendations. Read how we implemented these information into our project at <a href="https://2021.igem.org/Team:Aachen/Human_Practices">integrated human practices</a>. Despite the view that the TdT can also accept unnatural nucleotides, this must be observed for our specific kind of TdT and reaction buffer <a href="#citation2">2</a>. In summary, we had a particularly enriching conversation with some valuable tips and tricks for the implementation of our project.</p><div class="image"><img src="../assets/img/meeting_benner.jpeg" alt="Meeting with Steven Benner"><p>Figure 2: Meeting with Steven Benner</p></div><h1>Meeting with Jost Muth and Lena Julie Freund from the Fraunhofer IME Aachen</h1><p>To analyze our samples far more in detail, Sebastian Palluk suggested we to talk to the capillary electrophoresis (CE) experts from the Fraunhofer IME in Aachen. At the Fraunhofer IME they use an ABI3730 Genetic Analyzer to measure the length of for example PCR products. For this analysis method, our DNA had to be labeled with a fluorescent dye. Mrs. Freund suggested to use the 6FAM dye because of its low costs and good results. Through multiple questions, we concluded that the CE is an optimal analysis method for our samples because the DNA does not have to be double-stranded as the samples are denatured before analysis anyway. Furthermore, the size range from 35 nt to 500 nt within 1 nt resolution is the range of our synthesized fragments. After the questions, Mrs. Freund and Mr. Muth offered to support our project by trying to analyze our samples, help us with the interpretation of the data, and being open to questions in the future. Later, we had some more meetings to work out the analysis of our created DNA samples, which subsequently performed well. Our Modeling team used this data to adapt their model.</p><p>[//]: # Text wurde abgesegnet</p><h1>Meeting with Maximilian Schmidt from FZ Jülich</h1><p>Maximilian Schmidt is a member of the section IBG-4 Bioinformatics in FZ Jülich. We contacted him referring some questions about the Nanopore Sequencing with the MinION from Oxford Nanopore. For our progress in the lab, it is necessary to sequence the DNA strands we created, but there are some difficulties because of our specific method. So, we scheduled some meetings with Maximilian where we presented our project and talked about possible sequencing methods. He offered to join us in the lab and bring his MinION to analyze our samples with his help. It was exciting to learn about the sequence method of the MinION, and it was an excellent opportunity to ask as many questions about it as we wanted. Maximilian was very committed to sharing his experiences with us and was very helpful. At the end of the day, sequencing was successful, and we had a lot of new data for our project progress.</p><div class="image"><img src="../assets/img/Max Schmidt Bild.jpg" alt="Meeting Maximilian Schmidt in the lab in Aachen"><p>Figure 3: Meeting Maximilian Schmidt in the lab in Aachen</p></div><h1>Meetings with Elmar Weinhold</h1><p>Prof. Weinhold accompanied us on our path from the very beginning of our journey in February where we were still figuring out our topic. He is an organic chemist focusing on tools for genomics and proteomics.</p><p>Weinhold directly was enthusiastic about our project. He pointed out a few challenges we might be facing in course of our work in the wet lab. On our second meeting in March, we discussed the prevention of secondary structures and he advised us to use photolabile protecting groups (read more about it in the outlook). We also discussed the possibility of incorporating unnatural bases and elaborated that Deep Sequencing could be useful in this regard. He then helped us establish contact with Steven Benner. Quite at the end of our journey in September we also turned to Prof. Weinhold when we were struggling with our immobilization system. He advised us to try out the Streptavidin-Biotin magnetic immobilization, which ended up being our preferred system. Weinhold always met us with lots of positivity and enthusiasm and motivated us throughout the competition.</p><h1>Meeting with Dr. Florian Kraft</h1><p>Next-generation sequencing is a modern sequencing method, which enables massively parallel sequencing of numerous genes and has thus revolutionized genetic diagnostics in recent years. Dr. Florian Kraft, post-doc at the Institut for human genetic in the uni hospital in Aachen, deals precisely with New Sequencing techniques, especially Nanopore sequencing, which is our preferrable method to analyze our synthesized sequences.</p><p>Before our meeting and towards the end of our project we exchanged a lot of E-Mail’s and asked Dr. Kraft many questions about our samples for Nanopore sequencing, which he was always happy to answer. During our meeting in the middle of April, he explained the requirements for Nanopore-sequencing, such as the right primer length for Nanopore-sequencing. He raised the issue that using Homopolymers may be difficult because the nanopore could not detect the length of the polymer. Furthermore, we assessed the problem of adding his double-stranded nanopore adapters to our single-stranded DNA. Eventually, he proposed the opportunity to sequence our samples at the uni hospital at a low cost. We gladly accepted his offer and want to thank Dr. Kraft again for the great opportunity!</p><h1>Meeting with Dominik Heider and Anke Becker</h1><p>DNA data storages is a topical subject and attracts the attention of scientists from different fields. To drive innovation forward we believe it is crucial to meet and exchange ideas with people working on the same topic. For this reason, we met with Dominic Heider and Anke Becker, who are both involved in the project MOSLA (Molecular Storage for Longterm Archiving), at the beginning of June.</p><p>The project is designated to research of a DNA data storage. Dominic Heider is a bioinformatician working on Coding and especially on Error correction. Anke Becker is a Professor for Biology woking on the integration of unnatural bases in the storage code and on DNA-based Cloud-storages in different in vitro and in vivo formats. We discussed both of our research progresses. Even though the research group uses the Phosphoramidite synthesis instead of our enzymatic DNA syntheis system, Anke Becker could give us helpful advice for our wet lab. Prof. Becker took notice that on a microarray excess material could block the immobilized primer. Furthermore, she advised us to integrate an unnatural base pair to increase the data density and avoid secondary structures. It was a great opportunity to discuss our project with people working on the same subject.</p><h1>Meeting with Prof. Kalinowski and Dr. Rückert</h1><p>In the beginning of June, we turned to Prof. Jörn Kalinowski, who is an expert in field of Genomics and Sequencing from CeBiTech in Bielefeld, hoping to receive support concerning the analysis of samples via Nanopore seqeuencing. Kalinowksi joined the meeting with his lab coworker Dr. Christian Rückert, who is both an expert in Sequencing and an iGEM enthusiast. After listening to a presentation about our project, Kalinowski and Rückert explained detailly the principles of Nanopore sequencing to us. Firstly, they took notice that longer strands are preferred and everything under 200 base pairs is hard to detect. Then Rückert explained all the steps to us: preparing the strands with a base caller, attaching an adapter and a poly-A tail, phosphorylating the 5’ end. Nanopore sequencing is an important part of our project and this informative meeting gave us a clearer idea about its usage.</p><h1>Meeting with Prof. Doerte Rother, Dr. Thorsten Sehl, Jan-Dirk-Spoering</h1><p>In June we met with Prof. Doerte Rother, Dr. Thorsten Sehl and Jan Dirk-Spöring from FZ Jülich from the group of Biocatalysis. Dörte Rother is an expert in the field of enzyme immobilization and showed us possibilities which suit our purposes.</p><p>Back then, we have not decided whether we should immobilize a biotinylated primer on a Streptavidin plate, which is inspired by George Church, or use the new “Dip method”. Prof. Rother elaborated every immobilization method with us, showing us advantages and challenges. For the case that we choose the first approach, she warned us that immobilizing the enzyme by dialysis or any other kind of membrane could be disadvantageous because it can be blocked by the material fast. For the dipping method she advised us to immobilize biotinylated primer on a Streptavidin-coated magnetic bead and to use a magnetic stick. A few months later that turned out to be our preferred immobilization method. She also took notice that the efficiency of the enzyme could decrease after the immobilization. In conclusion, we had an informative and pleasant discussion.</p><div class="image"><img src="../assets/img/Human Practices/2021-06-11_Meeting-Rother.png" alt="Description of the image"><p>Figure 4: Description of the image</p></div><h1>Meeting with Dr. Sebastian Palluk</h1><p>Dr. Palluk is co-founder and CTO of Ansa Biotechnologies, a startup developing enzymatic DNA synthesis technologies. He himself participated in the iGEM competition himself and experienced that synthesizing DNA on your own was challenging. Now he works with a genetically modified TdT for biological applications and aims to improve the synthesis system for future implications. Mr. Palluk was probably one of our most important supporters.</p><p>We met with him three times in total and had frequent E-Mail contact during the whole project. In our first meeting, we started off by presenting our ideas and two approaches to encode data on DNA. Furthermore, we discussed four different ideas to prevent the formation of secondary structure of ssDNA and the affinity of the TdT for 3´OH base modifications. Mr. Palluk commented on our ideas on immobilization and explained briefly streptavidin immobilization on magnetic beads as well as sequencing techniques like Nanopore sequencing and capillary-electrophorese to view the length distribution. To conclude our first meeting, Mr. Palluk gave us a great overview of many important tasks we have to tackle in the future and encouraged us to give our best for the project. He belived that our unique interdisciplinary team will create a great project, but implementing everything we thought of would be harder than we thought.</p><p>In our second meeting with Mr. Palluk, at the beginning of May, we presented our possibilities to hinder secondary structures. He recommended starting our first experiments in solution with purification steps to establish our methods. Furthermore, he emphasized that it would be efficient to start working on the immobilization while conducting our first experiments. Mr. Palluk advised us to analyze the length of strands with gel electrophoresis and explained that driving PCR reaction beforehand is not necessary. Afterward, we discussed the possibilities for sequencing our strands, where he explained that sanger sequencing would only be possible if we would do a vector cloning beforehand.</p><p>In our third meeting, at the beginning of September, we discussed our ideas for building a hardware and explained the Maleimid-Cysteine binding system. Additionally, we examined the problem we have faced with our immobilization. With our current system, we would lose lots of Maleimid, which is too expensive for us. Mr. Palluk suggested using a Peptide-tag which would be immobilized the whole time and bind oligos with homologous sequences and overhangs. The TdT would bind at the overhangs and heat would remove it. Together, we thought of a „plan B“ for the immobilization. To create base transitions in solution we should add primer in excess, let the reaction finish until all nucleotides incorporated and add another nucleotide. To put it in a nutshell, Mr. Palluk was a very important expert for us who helped us create many solutions to our problem. Nobody knows the TdT better than he does. Thank you again Mr. Palluk for answering all our questions and E-Mails!</p><div class="image"><img src="../assets/img/Human Practices/2021-09-07_Meeting Palluk.png" alt="Description of the image"><p>Figure 5: Description of the image</p></div></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Yang, Z., Hutter, D., Sheng, P., Sismour, A. M., &amp; Benner, S. A. (2006)</p><cite>Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">34(21), 6095–6101</span></p><a class="in-text" href="https://doi.org/10.1093/nar/gkl633" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Artificially expanded genetic information system: A new base pair with an alternative hydrogen bonding pattern" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Tauraitė, D., Jakubovska, J., Dabužinskaitė, J., Bratchikov, M., &amp; Meškys, R. (2017)</p><cite>Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase</cite><p><span class="journalTitle">Molecules (Basel, Switzerland)</span> <span class="journalInfo">22(4)</span></p><a class="in-text" href="https://doi.org/10.3390/molecules22040672" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Modified Nucleotides as Substrates of Terminal Deoxynucleotidyl Transferase" target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="../assets/img/all_sponsors.png"></div></div></footer><script src="../content.bundle.js"></script></body></html>