<html lang="en"><head><meta charset="utf-8"/><meta content="width=device-width,initial-scale=1" name="viewport"/><title>Software | iGEM Aachen</title><script async="" type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
});</script><script async="" src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script>function resizeSidebar() {document.getElementById("contents").style.height = (document.getElementById("list").scrollHeight + 100 + "px"); }
window.onresize = resizeSidebar;</script><link href="https://2021.igem.org/Template:Aachen/css/contentCSS?action=raw&amp;ctype=text/css" rel="stylesheet"/></head><body data-offset="150" data-spy="scroll" data-target="#contentnav" onload="resizeSidebar()"><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://2021.igem.org/Team:Aachen"><img src="https://2021.igem.org/wiki/images/1/14/T--Aachen--img--teamlogo_2021.svg"/></a><button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown" role="button">Project</a><div aria-labelledby="navbarProjectDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Description">Description</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Implementation">Implementation</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Engineering">Engineering Success</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Contribution">Contribution</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Outlook">Outlook</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarWetlabDropdown" role="button">Wetlab</a><div aria-labelledby="navbarWetlabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Experiments">Experiments</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Notebook">Notebook</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Protocols">Protocols</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Results">Results</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Safety">Safety</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarDrylabDropdown" role="button">Drylab</a><div aria-labelledby="navbarDrylabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Hardware">Hardware</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Model">Modeling</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Software">Software</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarHuman PracticesDropdown" role="button">Human Practices</a><div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Communication">Science Communication</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Collaborations">Collaborations</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Memes">Memes</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarTeamDropdown" role="button">Team</a><div aria-labelledby="navbarTeamDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Members">Members</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="https://2021.igem.org/wiki/images/a/a8/T--Aachen--img--headers--Software.jpg"/><img class="header-text" src="https://2021.igem.org/wiki/images/7/73/T--Aachen--img--headers--header_texts--Software.svg"/><img class="header-effect" src="https://2021.igem.org/wiki/images/d/d7/T--Aachen--img--headers--header_effect.svg"/></header><nav class="bg-image" id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents" style="flex-direction: column"><h5>Software</h5><ul id="list"></ul></div></div><div class="content col-lg-9"><article><h1>Purpose of Our Software</h1><p>The goal of Storagene is to store data on DNA. To do this, we first need to encode data in form of a base sequence. Even more importantly, we need to be able to decode a sequenced DNA strand to recover the original data. These two tasks are performed by our software. The main focus lies in error correction in the decoding process. Many errors can occur during DNA synthesis and sequencing. By utilizing the peculiarities of our semi-specific synthesis in combination with the capabilities of nanopore sequencing, we will be able to robustly decode data from DNA. This plays an important role in making this technique easier to use and thus accessible to everyone. Our software addresses a problem that is not very common today. However, it has the potential to become a standard bioinformatics tool when DNA data storage is eventually technically integrated into everyday life.</p><h1>Overview</h1><p>Our software is written in Python. Right now, it consists of a program library and a command-line tool that utilizes the library. The command-line tool makes all the basic functionalities of our library available, namely encoding and decoding data. In addition, it would be possible to implement a graphical user interface, which would make the usage even easier for everyone.</p><p>In the following, the two functions of our software encoding and decoding are explained in detail. It will become clear how we can store data on DNA, although our synthesis approach is semi-specific. We will also discuss the challenges that arise when decoding DNA strands.</p><h1>Encoding Data into DNA</h1><div class="image"><img alt="Overview of the steps needed for encoding data into DNA." src="https://2021.igem.org/wiki/images/5/5d/T--Aachen--img--Software--encoding.svg" style="width: 60%"/><p style="font-size:1rem; text-align:center"><b>Figure 1</b>: Overview of the steps needed for encoding data into DNA.</p></div><h2>Changes of Nucleotides as Information Carriers</h2><p>Our DNA synthesis is semi-specific, meaning that we cannot control how many nucleotides are attached in each step. This means that we will always have chunks of an unknown length of each nucleotide, e.g. "AAACCCCCAAAAAGGGG". These chunks are also called homonucleotides. Nevertheless, to be able to encode data in this, we are using a rather unusual encoding scheme. Instead of encoding one bit of information to each base pair, we are encoding information in each change of bases. For example, the change from a homonucleotide of <strong>A</strong>s to a homonucleotide of <strong>C</strong>s could encode a <code>1</code>.</p><h2>Using a Ternary Code</h2><p>From each homonucleotide, three other homonucleotides are reachable. From "A" we can go to "C", "G", and "T". From "C" we can go to "A", "G", and "T", and so on. So, to be able to use all possible combinations, we do not need a binary code, but a ternary code. Computers work with a binary code, meaning that every information is encoded in bits. Each bit can either be <code>0</code> or <code>1</code>. Instead, in our ternary code information is encoded in trits. Each trit can have the states <code>0</code>, <code>1</code>, or <code>2</code>. We adapted both the idea of encoding information in base changes and the use of a ternary system from Church et al. <a href="#citation1">1</a>.</p><h2>Converting from Binary to Ternary</h2><p>Our software can encode arbitrary files into DNA strands. Therefore, the first step is to convert binary data into ternary data. Computers work with bytes, so chunks of eight bits. Hence, it is natural to first split the string of bits we get from a file into bytes. Then, the software converts each byte into trits.</p><p>The count of values a number can take is calculated by the formula $b^l$. In this $b$ is the base of the number system, e.g., $2$ for the binary system, and the superscript 1 is the number of digits. Since a byte consists of eight bits with two states each, every byte can have $2^8=256$ states. To represent the information of one byte with trits, we need enough trits to represent at least $256$ different states. Five trits can represent $243$ states, since $3^5=243$. So obviously, five trits are not enough to encode one byte. Six trits, in contrast, represent $3^6=729$ states. So one byte can be encoded with six trits. In the following, we will call a sequence of six trits a tryte. However, this also shows that converting one byte into one tryte is extremely inefficient. $\frac{256}{729} \approx 35.12$%, so roughly one-third of the actual capacity of this code is used.</p><h2>Converting Number Systems More Efficiently</h2><p>Instead of converting eight bits into six trits, it is also possible to cut up the original data differently. In the best case, this can lead to a more efficient conversion. For example, three bits could be converted into two trits. Since $2^3=8$ and $3^2=9$, $\frac{8}{9}$ or 89% of the maximum capacity is used. And even more efficient conversion would be from eleven bits to seven trits with a capacity of 94%. However, both of these conversions would also require some additional computation. Arbitrary bit strings usually cannot be divided into chunks of three or eleven bits. Therefore, the last chunk would need to be padded with bits. This means that the encoded data also needs to contain the information of how many bits were padded to the end. Due to this additional complexity, other conversions than from byte to tryte are not implemented in our software right now.</p><h1>Decoding and Error Correction in Sequencing Data</h1><div class="image"><img alt="Overview of the steps for retrieving information from DNA." src="https://2021.igem.org/wiki/images/8/81/T--Aachen--img--Software--decoding.svg" style="width: 60%"/><p style="font-size:1rem; text-align:center"><b>Figure 2</b>: Overview of the steps for retrieving information from DNA.</p></div><p>As you can see in figure 2, decoding mainly works just as encoding but backward. However, one additional step is necessary before data can be decoded. From the sequenced data, we first must retrieve the original DNA sequence that was used for synthesis. To do so, we must understand what happens in synthesis. As discussed before, an unknown number of nucleotides is attached in each step. This is due to the semi-specific approach. In addition to that, various primers and other sequences are added to both ends of the encoded sequence. This partly happends during synthesis and partly in preparation for sequencing.</p><div class="image"><img alt="Example of DNA sequences in a fastq file." src="https://2021.igem.org/wiki/images/6/6e/T--Aachen--img--Software--fastq_example.jpg" style="width: 100%"/><p style="font-size:1rem; text-align:center"><b>Figure 3</b>: Example of DNA sequences in a fastq file.</p></div><p>The decoding starts with a <code>fastq</code> file containing sequencing data. Figure 3 shows an example of such a file. Each read from sequencing is expressed by four lines. While the first and third lines can contain an identifier and description of the sequence, the second and fourth lines are more important to us. The second line contains the actual DNA sequence. In line 4 the quality of the sequence is encoded. This is important to assess how reliable the sequencing was. The quality is encoded in values reaching from 0x21 (33 in decimal) to 0x7E (126) and displayed as the corresponding ASCII character. So "!" marks the lowest quality, while "~" stands for the highest quality.</p><h2>Removing Primers and Cleaning Up Data</h2><p>The first step necessary for decoding is the removal of the primer since this does not contain any relevant information. To do so, the software scans every sequence for a subsequence that is similar to the primer. If it finds the primer in a sequence, it deletes the primer and everything prior to it from the sequence. If it does not find a primer, it deletes the whole sequence from the dataset. As shown in figure 4, the sequence encoding information begins right behind the primer. So by removing the primer and everything before it, we know exactly that the sequence now begins with the encoding sequence. After that, this step is repeated with the reverse primer. The reverse primer starts directly behind the encoding sequence. After removing it and everything behind it, all sequences now only contain the relevant information.</p><div class="image"><img alt="Overview of the components of a sequence." src="https://2021.igem.org/wiki/images/5/54/T--Aachen--img--Software--sequence_overview.svg" style="width: 90%"/><p style="font-size:1rem; text-align:center"><b>Figure 4</b>: Overview of the components of a sequence.</p></div><p>Before the primer, there is an additional sequence added. This is necessary for nanopore sequencing. By clearing the sequences in the way described before, these are also removed automatically. If there is any sequence not containing one of the primers, it will be removed completely. This ensures that no corrupted data disturb the further steps.</p><h3>Finding Primer Sequences with Errors</h3><p>When searching for primers in the sequences, not only perfect matches should be found. In synthesis and sequencing, errors can happen in various places. So all subsequences that are <em>very similar</em> to a primer should be classified as such. For that classification, the Levenshtein distance is used. This is a very popular measure for all kinds of string comparisons. It describes how many insertions, deletions, or substitutions of symbols are needed to match two strings. For example, the sequences "ACGT" and "ACGA" have a Levenshtein distance of 1, since one substitution is needed to match them. In the same manner, the sequences "ACGT" and "ACGTA" also have a distance of 1, since they can be matched with one insertion in the first sequence.</p><h2>Merging the Sequences</h2><p>After removing additional parts from the sequences, only the relevant parts should be left. Now the obvious next step would be to take a sequence, merge all homonucleotides into single nucleotides and go on with decoding. However, many potential sources of error make it unlikely that every sequence contains the full and correct information. To compensate for that, our software compares all remaining sequences to find the most likely original sequence. First, every sequence gets transformed into a sequence of homonucleotides internally. Each homonucleotide holds information about the nucleotide of which it is composed, but also its length and average quality. This is visualized in figure 5. Then, the first homonucleotides of all sequences are compared to find the most likely one. For that, the lengths and average qualities are used to find the most likely nucleotide. This step is repeated multiple times to recreate the whole sequence. Usually, the sequences do not all have the same lengths. Many sequences only contain a few homonucleotides, while some are much longer.</p><div class="image"><img alt="Overview of the representation of a sequence as homonucleotides." src="https://2021.igem.org/wiki/images/8/85/T--Aachen--img--Software--homonucleotide_overview.svg" style="width: 90%"/><p style="font-size:1rem; text-align:center"><b>Figure 5</b>: Overview of the representation of a sequence as homonucleotides.</p></div><p>To decide when to stop comparing the remaining sequences, a percentile is used. For example, the percentile can be 50%. That means that the sequences are compared as long as at least 50% of all sequences reach that lengths. If 50 out of 100 sequences contain six homonucleotides and all others are shorter, for example, the first six homonucleotides are compared.</p><p>After that step, the software saves the recreated sequence in a <code>fasta</code> file. This is similar to the fastq file mentioned before but does not contain a quality measure. The sequence in that file can then be used to decode the original information again. As said before, this works in the same manner as encoding but backward.</p><h1>Application as Demonstration for Integrated Human Practices</h1><p>As described on our <a href="https://2021.igem.org/Team:Aachen/Software/Human_Practices#ConversationattheCityArchiveinAachen">Human Practices page</a>, we reached out to the city archive in Aachen to discuss potential use cases of our proposed storage solution. The city archive has digitalized many historical documents like newspapers. As a technology demonstration, we encoded a scan of a newspaper article from 1933.</p><div class="image"><img alt="Demonstration of our encoding software." src="https://2021.igem.org/wiki/images/e/ec/T--Aachen--img--Software--EchoderGegenwart_1933.png" style="width: 100%"/><p style="font-size:1rem; text-align:center"><b>Figure 6</b>: Demonstration of our encoding software.</p></div><p>Figure 4 shows a clipping of the newspaper page as well as an extract from the encoded sequence. With the link below, you can download the full sequence as a fasta file.</p><a class="extraLinks" href="https://2021.igem.org/wiki/images/a/ac/T--Aachen--img--Software--EchoderGegenwart_1933_jpg_fasta.zip">Echo der Gegenwart</a></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Lee, H. H., Kalhor, R., Goela, N., Bolot, J., &amp; Church, G. M. (2019).</p><cite>Terminator-free template-independent enzymatic DNA synthesis for digital information storage.</cite><p><span class="journalTitle">Nature Communications</span> <span class="journalInfo">10(1).</span></p><a class="in-text" href="https://doi.org/10.1038/s41467-019-10258-1" rel="noopener" target="_blank">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Terminator-free template-independent enzymatic DNA synthesis for digital information storage." rel="noopener" target="_blank">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="https://2021.igem.org/wiki/images/a/af/T--Aachen--img--all_sponsors.png"/></div></div></footer><script src="https://2021.igem.org/Template:Aachen/content-bundleJS?action=raw&amp;ctype=text/javascript"></script></body></html>