<html lang="en"><head><meta charset="utf-8"/><meta content="width=device-width,initial-scale=1" name="viewport"/><title>Software | iGEM Aachen</title><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
});</script><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="https://2021.igem.org/Template:Aachen/css/contentCSS?action=raw&amp;ctype=text/css" rel="stylesheet"/></head><body data-offset="150" data-spy="scroll" data-target="#contentnav"><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://2021.igem.org/Team:Aachen"><img src="https://2021.igem.org/wiki/images/1/14/T--Aachen--img--teamlogo_2021.svg"/></a><button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown" role="button">Project</a><div aria-labelledby="navbarProjectDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Description">Description</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Implementation">Implementation</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Engineering">Engineering</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Contribution">Contribution</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Outlook">Outlook</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Memes">Memes</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarWetlabDropdown" role="button">Wetlab</a><div aria-labelledby="navbarWetlabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Experiments">Experiments</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Notebook">Notebook</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Protocols">Protocols</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Results">Results</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Safety">Safety</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarDrylabDropdown" role="button">Drylab</a><div aria-labelledby="navbarDrylabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Hardware">Hardware</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Model">Modeling</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Software">Software</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarHuman PracticesDropdown" role="button">Human Practices</a><div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Science_Communication">Science Communication</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarTeamDropdown" role="button">Team</a><div aria-labelledby="navbarTeamDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Members">Members</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="https://2021.igem.org/wiki/images/a/a8/T--Aachen--img--headers--Software.jpg"/><img class="header-text" src="https://2021.igem.org/wiki/images/7/73/T--Aachen--img--headers--header_texts--Software.svg"/><img class="header-effect" src="https://2021.igem.org/wiki/images/d/d7/T--Aachen--img--headers--header_effect.svg"/></header><nav id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Purpose of Our Software</h1><p>The goal of Storagene is to store information on DNA. To do this, we first need to encode information in form of a base sequence. Even more importantly, we need to be able to decode a sequenced DNA strand to recover the original information. These two tasks are performed by our software. The main focus lies on error correction in the decoding process. Many errors can occur during DNA synthesis and sequencing. By utilizing the peculiarities of our semi-specific synthesis in combination with the capabilities of nanopore sequencing, we will be able to robustly decode information from DNA. This plays an important role in making this technique easier to use and thus accessible to everyone. Our software addresses a problem that is not very common today. However, it has the potential to become a standard bioinformatics tool when DNA data storage is eventually technically integrated into everyday life.</p><h1>Overview</h1><p>Our software is written in Python. Right now, it consists of a program library and a command line tool that utilizes the library. The command line tool makes all the basic functionalities of our library available, namely encoding and decoding information. In addition, it would be possible to implement a graphical user interface, which would make the usage even easier for everyone.</p><p>In the following, the two functions of our software encoding and decoding are explained in detail. It will become clear how we are able to store information on DNA, although our synthesis approach is semi-specific. We will also discuss the challenges that arise when decoding information.</p><h1>Encoding Information on DNA</h1><div class="image"><img alt="Overview over the steps needed for encoding data into DNA." src="https://2021.igem.org/wiki/images/5/5d/T--Aachen--img--Software--encoding.svg"/><p>Figure 1: Overview over the steps needed for encoding data into DNA.</p></div><h2>Changes of Nucleotides as Information Carriers</h2><p>Our DNA synthesis is semi-specific, meaning that we cannot control how many nucleotides are attached in each step. This means that we will always have chunks of unknown length of each nucleotide, e.g. "AAACCCCCAAAAAGGGG". These chunks are also called homonucleotides. Nevertheless, to be able to encode information in this, we are using a rather unusual encoding scheme. Instead of encoding one bit of information to each base pair, we are encoding information in each change of bases. For example, the change from a homonucleotide of <strong>A</strong>s to a homonucleotide of <strong>C</strong>s could encode a <code>1</code>.</p><h2>Using a Ternary Code</h2><p>From each homonucleotide, three other homonucleotides are reachable. From "A" we can go to "C", "G", and "T". From "C" we can go to "A", "G", and "T", and so on. So, to be able to use all possible combinations, we do not need a binary code, but a ternary code. Computers work with a binary code, meaning that every information is encoded in bits. Each bit can either be <code>0</code> or <code>1</code>. Instead, in our ternary code information is encoded in trits. Each trit can have the states <code>0</code>, <code>1</code>, or <code>2</code>.</p><h2>Converting form Binary to Ternary</h2><p>Our software is able to encode arbitrary files into DNA strands. Therefore, the first step is to convert binary information into ternary information. Computers work with bytes, so chunks of eight bits. Hence, it is natural to first split the string of bits we get from a file into bytes. Then, the software converts each byte into trits.</p><p>The count of values a number can take is calculated by the formula $b^l$. In this $b$ is the base of the number system, e.g., '2' for the binary system, and$^l$ is the number of digits. Since a byte consists of eight bits with two states each, every byte can have $2^8=256$ states. To represent the information of one byte with trits, we need enough trits to represent atleast $256$ different states. Five trits can represent $243$ states, since $3^5=243$. So obviously, five trits are not enough to encode one byte. Six trits, in contrast, represent $3^6=729$ states. So one byte can be encoded with six trits. In the following, we will call a sequence of six trits a tryte. However, this also shows that converting one byte into one tryte is extremely inefficient. $\frac{256}{729} \approx 35.12$%, so roughly one third of the actual capacity of this code is actually used.</p><h2>Converting Number Systems More Efficiently</h2><p>Instead of converting eight bits into six trits, it is also possible to cut up the original data differently. In the best case, this can lead to a more efficient conversion. For example, three bits could be converted into two trits. Since $2^3=8$ and $3^2=9$, $\frac{8}{9}$ or 89% of the maximum capacity are actually used. And even more efficient conversion would be from eleven bits to seven trits with a capacity of 94%. However, both of these conversions would also require some additional computation. Arbitrary bit strings usually cannot be divided into chunks of three or eleven bits. Therefore, the last chunk would need to be padded with bits. This means that the encoded information also needs to contain the information of how many bits were padded to the end. Due to this additional complexity, other conversions than from byte to tryte are not implemented in our software right now.</p><h1>Decoding Information from Sequencing Data</h1><div class="image"><img alt="Overview over the steps for retrieving information from DNA." src="https://2021.igem.org/wiki/images/8/81/T--Aachen--img--Software--decoding.svg"/><p>Figure 2: Overview over the steps for retrieving information from DNA.</p></div></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="https://2021.igem.org/wiki/images/a/af/T--Aachen--img--all_sponsors.png"/></div></div></footer><script src="https://2021.igem.org/Template:Aachen/content-bundleJS?action=raw&amp;ctype=text/javascript"></script></body></html>