<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Experiments | iGEM Aachen</title><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../css/content.css" rel="stylesheet"></head><body data-spy="scroll" data-target="#contentnav" data-offset="100"><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="/"><img src="../assets/img/teamlogo_2021.svg"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="/Description">Description</a><a class="dropdown-item" href="/Implementation">Implementation</a><a class="dropdown-item" href="/Engineering">Engineering</a><a class="dropdown-item" href="/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="/Contribution">Contribution</a><a class="dropdown-item" href="/Outlook">Outlook</a><a class="dropdown-item" href="/Memes">Memes</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarWetlabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Wetlab</a><div class="dropdown-menu" aria-labelledby="navbarWetlabDropdown"><a class="dropdown-item" href="/Experiments">Experiments</a><a class="dropdown-item" href="/Notebook">Notebook</a><a class="dropdown-item" href="/Protocols">Protocols</a><a class="dropdown-item" href="/Results">Results</a><a class="dropdown-item" href="/Safety">Safety</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarDrylabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Drylab</a><div class="dropdown-menu" aria-labelledby="navbarDrylabDropdown"><a class="dropdown-item" href="/Hardware">Hardware</a><a class="dropdown-item" href="/Model">Modeling</a><a class="dropdown-item" href="/Software">Software</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="/Human_Practices">Human Practices</a><a class="dropdown-item" href="/Communication">Communication</a><a class="dropdown-item" href="/Education">Education</a><a class="dropdown-item" href="/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="/Team">Team</a><a class="dropdown-item" href="/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="../assets/img/headers/Experiments.jpg"><img class="header-text" src="../assets/img/headers/header_texts/Experiments.svg"><img class="header-effect" src="../assets/img/headers/header_effect.svg"></header><nav id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Overview</h1><p>The goals of our experiments were the following:</p><ul><li>Characterizing the terminal transferase (TdT) reaction and showing that TdT can incorporate nucleotides to ssDNA primers</li><li>Immobilizing our ssDNA primers</li><li>Performing a cyclic semi-specific synthesis, in which an unknown number of one nucleotide type is attached in each reaction cycle</li><li>Proving that this semi-specific synthesis works and that data can be encoded and recovered in this way by sequencing the created fragments</li><li>Investigating the problem of secondary structures during strand elongation</li></ul><p>Our lab notebook is available <a href="https://2021.igem.org/Team:Aachen/Notebook">here</a>.</p><p>Detailed protocols can be found <a href="https://2021.igem.org/Team:Aachen/Protocols">here</a>.</p><p>The results are summarized <a href="https://2021.igem.org/Team:Aachen/Results">here</a>.</p><h1>TdT Tailing Reaction</h1><p>The first step was to test whether TdT binds nucleotides to an ssDNA primer. A standard TdT tailing reaction was performed and verified by <a href="#GelElectrophoresis">gel electrophoresis</a>. To better characterize the reaction, many parameters such as incubation time and temperature as well as nucleotide, primer, cofactor, and enzyme concentration were varied. Different nucleotides and combinations of those were tried to reveal significant differences in incorporation speed. This information was crucial for building hardware that can attach a roughly equal amount of each nucleotide, following the principle of: “As little as possible and as much as necessary“. This means that the goal is to find conditions where each strand is elongated as little as possible, but with at least one nucleotide in each cycle (see <a href="https://2021.igem.org/Team:Aachen/Modeling">modeling</a>). For this purpose, the length distribution of the synthesized fragments was analyzed by <a href="#CapillaryElectrophoresis">capillary electrophoresis</a>.</p><p>Furthermore, different TdTs were tested. In the first experiments, we used a recombinant TdT from NEB, but all other experiments were performed with a recombinant TdT from Thermo Fischer Scientific. They use very different reaction buffers, which significantly changed the incorporation speed and nucleotide preferences of TdT. All our summarized results concern the Thermo Fisher TdT.</p><h1>Cyclic Synthesis and Primer Immobilization</h1><p>A cyclic synthesis in solution was tried by just adding the next type of nucleotide to the solution after each elongation. We used a higher primer concentration than in our standard reaction and increased the nucleotide concentration in each reaction step. For the semi-specific synthesis, the TdT reaction in solution has the decisive disadvantage that the different nucleotides mix after the first few reaction cycles because there still is a nucleotide excess after each reaction step with that specific nucleotide. Hence, it is not ensured that the added nucleotide is also the incorporated nucleotide and synthesis errors accumulate.</p><p>Therefore, the sample would have to be purified after each step to isolate the extended ssDNA strands and lose excess nucleotides. Unfortunately, after purification with standard purification kits, another TdT tailing reaction does not work (see <a href="https://2021.igem.org/Team:Aachen/Results">results</a>).</p><p>Immobilizing the primers can solve this problem and is the basis for a cyclic synthesis of longer strands. Current techniques involve immobilization on a streptavidin-coated surface, for instance in a flowcell with different mastermixes <a href="#citation1">1</a>. For our purpose, a novel and simple method was developed: The primers were immobilized on a stick and dipped into the reaction solutions with the different nucleotides. Various immobilization systems were tried:</p><ol><li>Primers with amino modification and anchor peptide LCI</li><li>Primers with maleimide modification and anchor peptide LCI</li><li>Biotinylated primers and streptavidin-coated magnetic beads</li></ol><div class="image"><img src="../assets/img/Experiments/Experiments--Amino-C6.png" alt="Immobilization of amino-labeled TdT reaction primers via anchor peptide fusion protein."><p>Figure 1: Immobilization of amino-labeled TdT reaction primers via anchor peptide fusion protein.</p></div><div class="image"><img src="../assets/img/Experiments/Experiments--maleimid.png" alt="Immobilization of maleimide-labeled TdT reaction primers via anchor peptide fusion protein."><p>Figure 2: Immobilization of maleimide-labeled TdT reaction primers via anchor peptide fusion protein.</p></div><p>For the first two methods, we used fusion proteins from the Leibniz Institute of Interactive Materials (DWI). The amino oligo can bind to a carboxyl functional group in the fusion protein in an EDC mediated reaction and the maleimide oligo binds to the thiol functional group of a cysteine in the fusion protein (see figure 1 and 2). Those conjugates, consisting of the modified primer and the anchor peptide fusion protein, were bound to a polystyrene stick.</p><div class="image"><img src="../assets/img/Experiments/Experiments--streptavidin-biotin.png" alt="Immobilization of biotin-TEG-labeled TdT reaction primers via streptavidin-coated magnetic beads on rod magnet."><p>Figure 3: Immobilization of biotin-TEG-labeled TdT reaction primers via streptavidin-coated magnetic beads on rod magnet.</p></div><p>For the third method, we incubated our biotinylated primers with streptavidin-coated magnetic beads. We used biotin with the spacer TEG to minimize steric hindrance (see figure 3). The beads were immobilized with a rod magnet.</p><h2>The DIP Method</h2><p>To elongate the DNA strand in a controlled manner, we used the third immobilization system with the streptavidin-coated magnetic beads and the rod magnet and developed the DIP method:</p><p>Four different Eppendorf tubes containing the TdT reaction mixture and one nucleotide were prepared. Another tube contained the biotinylated primer. The beads were added to the latter so that the primer could bind to them. Then, a strong bar magnet surrounded by a plastic envelope was dipped into the tube. The magnetic beads with the biotinylated primer could now be transferred to the first reaction tube. The bar magnet was removed from the plastic envelope so that the magnet beads were released into the reaction solution. After the reaction, the bar magnet was placed back into the plastic envelope, transferring the beads to a wash tube with water to lose excess nucleotides. It was then dipped into the next reaction solution and the above steps were repeated until the strand reached the desired length and sequence. The reaction time was varied in different experiments to find out how many nucleotides were attached.</p><h1>Gel Electrophoresis</h1><p>For first sample analysis, our method of choice was the agarose gel electrophoresis because of easy handling and fast results. The final target was to sequence our self-generated DNA fragments, but sequencing methods had to be established first. As the first samples were generated with only one type of nucleotide, length determination was most important. For our samples (low concentrated very short ssDNA fragments, ~20-1000 nt), the best length analysis method (apart from <a href="#CapillaryElectrophoresis">capillary electrophoresis</a>) would typically be a denaturing PAGE, which was tried but rejected because of inconsistent results and performance difficulties. We lost a lot of time at the beginning of our project due to the optimization attempts, as making our DNA strands visible was the first critical step. At last, we found conditions for an agarose gel electrophoresis that enabled us to clearly see ssDNA samples (see <a href="https://2021.igem.org/Team:Aachen/Engineering">engineering success</a>). A 2.5% agarose solution in TBE buffer was used. The gel was stained with SYBR gold solution for 20 min. A dsDNA ladder was used because we did not need a precise length determination. These conditions were not optimal but sufficed for a first rough assessment of the samples, which would then undergo other analysis steps.</p><h1>Capillary Electrophoresis</h1><p>We had the possibility to let our samples be analyzed with a fragment analyzer at Fraunhofer Institute IME. This analysis method has several advantages over gel electrophoresis: Fragments from 35 bp to 500 bp can be sized to within 1 bp resolution and can be polymorphic in their lengths <a href="#citation2">2</a>. In addition, ssDNA can be analyzed.</p><p>So this was the perfect method to detect small differences in length and establish the length distribution for different nucleotides and conditions.</p><p>Of course, this is no sequencing method and it cannot tell, which nucleotides are incorporated. Furthermore, one needs primers labeled with a fluorescent dye to be detected by the fragment analyzer. It was therefore not possible to analyze immobilized DNA because it was already labeled with the immobilization group.</p><p>The TdT reaction with the fluorescent-labeled primers was performed in the same way as with the unlabeled primers. We used primers labeled with 6-FAM.</p><h1>PCR</h1><p>It is difficult to sequence ssDNA strands, in particular those with polymorphic length. So we wanted to develop a PCR system that would enable us to amplify polymorphic single strands with an unknown sequence and make them double-stranded in the process. A standard PCR utilizes the fact that the specific sequence to be amplified is well-known. Complementary PCR primers can be designed. We only know the sequence of the beginning of our strand because it belongs to our self-designed TdT reaction primer. Due to the semi-specific synthesis, however, the exact sequence of attached nucleotides is unknown.</p><p>We propose a system in which the reverse primer for the PCR has a poly-T sequence. Now we only have to make sure, that a poly-A sequence is attached at the end of the strand. Furthermore, we designed the PCR primers with an overhang that contains a restriction site for cloning purposes.</p><div class="image"><img src="../assets/img/Experiments/Experiments--PCR.gif" alt="Operating principle of a PCR with a single-stranded template and poly-T reverse primer."><p>Figure 4: Operating principle of a PCR with a single-stranded template and poly-T reverse primer.</p></div><p>Problems with this system are that the poly-T primer not only binds to the very end of, but to any poly-A sequence in the strand. One could avoid using dATP to elongate the strand and only use it for poly-A tailing, but this would limit the data capacity for encoding information. Furthermore, the poly-T primer has a low annealing temperature, favoring unspecific PCR products <a href="#citation3">3</a>.</p><p>The first PCR primers we designed were suboptimal because they were very short and thus had a low annealing temperature. We later designed longer primers, namely a 50 nt poly-T primer and a 25 nt forward primer, which then had a similar melting temperature and an annealing temperature of 63 °C.</p><p>We worked with the proofreading Phusion High-Fidelity DNA Polymerase from Thermo Fisher Scientific.</p><h1>Sequencing</h1><h2>Cloning and Sanger Sequencing</h2><p>As our synthesized DNA fragments are polymorphic, Sanger sequencing did not seem to be an option at first. But as it is fast, cost-effective, and easy to evaluate for short DNA fragments, we developed a system that enabled us to sequence random individual fragments out of the polymorphic batch.</p><p>A PCR was performed and the product was treated as an insert for a cloning reaction. We used E. coli Top10 and E. coli DH10B (NEB 10-beta) and tried the GeneArt Seamless Cloning and Assembly Kit with the GeneArt pMA Vector, the Promega pGEM-T Easy Vector System, and the Invitrogen TOPO TA Cloning Kit. 10-20 colonies were picked and a colony PCR was performed. The PCR product was then purified to obtain one sample strand from the polymorphic batch. Several strands were analyzed to provide an overview of the synthesized fragments.</p><h2>Nanopore Sequencing</h2><p>Nanopore sequencing is typically not used for short DNA fragments and is especially not suited for strands containing many homopolymeric regions due to its error rate <a href="#citation4">4</a>. But it is particularly suitable for polymorphic DNA fragments, which is why we choose it to sequence the synthesized strands. It generates large amounts of data, which were difficult to evaluate, but enabled us to build a proof of concept.</p><h1>Preventing Secondary Structures</h1><p>In addition to its affinity for chemical and nucleolytic attack, ssDNA is thermodynamically more unstable than dsDNA, resulting in the formation of duplex secondary structures such as hairpins <a href="#citation5">5</a>. TdT accessibility to the 3’-OH end can be restricted. Since TdT always needs at least a three-nucleotide overhang to be able to attach nucleotides, synthesis can be inhibited or come to a halt if unfavorable secondary structures form <a href="#citation6">6</a>.</p><p>We thought of new methods to prevent the formation of secondary structures.</p><h2>Single-Strand DNA-Binding Proteins (SSBs)</h2><p>One idea was to add single-strand DNA-binding proteins (SSBs) to the TdT reaction. SSBs are proteins that bind to ssDNA, normally during replication, recombination, or repair processes, stabilizing it and preventing reannealing <a href="#citation7">7</a>. For our purpose, it was important that the SSB stabilizes the ssDNA strand, but also does not inhibit the TdT reaction by binding too tightly.</p><p>We used the Single-Stranded DNA Binding Protein from Thermo Fisher Scientific, which is a monomer consisting of 4 identical subunits, each with a size of 18.9 kDa. 8 - 16 nucleotides are bound per monomer. <a href="#citation8">8</a> Different SSB concentration were applied to TdT reactions. Strands synthesized with and without SSBs were sequenced to find out if longer strands are generated with SSBs.</p><p>There are various other SSBs from different organisms with different properties. Due to a lack of time and because only few are commercially available, we did not get to try other SSBs.</p><h2>Other Methods</h2><p>Other methods included adding short oligos to stabilize the strand by random local hybridizations and using unnatural bases. Unfortunately, we did not have the chance to properly try them in the lab. (see <a href="https://2021.igem.org/Team:Aachen/Outlook">outlook</a>)</p><h1>Oligo Sequences</h1><p>We designed most of the primers and oligos for our purposes ourselves. Here are the sequences:</p><h2>TdT Reaction Primers</h2><p>Our very first TdT reactions were performed with small leftover ssDNA primers from the lab.</p><p>GALI fwd: 5’-ATTTTCGGTTTGTATTACTTC-3‘</p><p>GALI rev: 5’-GTTCTTAATACTAACATAACT-3‘</p><p>GALI0 fwd: 5’-GGTGGTAATGCCATGTAATATG-3‘</p><p>GALI0 rev: 5’-GGCAAGGTAGACAAGCCGACAAC-3‘</p><p>We then originally designed two different TdT Reaction Primers, one with an AT-rich sequence and the other with a GC-rich sequence, to investigate the formation of secondary structures because a high amount of complementary bases favors secondary structure formation [citation needed]. The sequences are completely random. We also ordered these oligos with different 5’-modifications for analysis and immobilization purposes.</p><p>AT-rich Primer:<br><code>5‘-ACAATTCATCTTAGACTGACTATTATAAATCACACAGAGAAATCTATCAGAATTCTAGATATTCATTAAGTAATTTGAGTAGTTAATTTAAGCTTAATAA-3‘</code></p><p>GC-rich Primer:<br><code>5’-GGACTTTTGCGCGTAGGCGGCGCGCGGCGGCCGCGGCCCGGGCCGCGGCCCGGGCCGGCTGGGTGGAGAGCCTACGGAGAGCTAATGCTAGCGTGCTGTAG-3‘</code></p><p>Modifications: 5‘ 6-FAM for capillary electrophoresis, 5‘ Amino Modifier C6 for immobilization, 5‘ Maleinimid for immobilization, 5‘ Biotin-TEG for immobilization.</p><p>As we were not able to perform many experiments concerning secondary structures and also had problems with the PCR reaction, we later designed a new primer with 50% GC-content and also a random sequence. We only ordered this primer with the 5‘ Biotin-TEG modification for immobilization.</p><p>Normal Primer:<br><code>5‘-ACAATTCATCTTAGTGCCAGACTACTGTACTGTCCCAGAGAATAGACTTGCTTCGTGCGCACCCGTATGCCGCTCGATATGCGTAGCCGTCCATGTTGAT-3‘</code></p><h2>PCR Primers</h2><p>Our first PCR primers were very short and had a low annealing temperature; they did not work well. We designed them with an overhang that contains restriction cut sites for SacI (forward primer) and KpnI (reverse Primer) for both the AT-rich and GC-rich TdT reaction primers.</p><p>AT SacI fwd: 5’-GAGCTCACAATTCATCTTAG-3’</p><p>AT KpnI rev: 5’-GGTACCTTTTTTTTTTTTTTTTT-3’</p><p>GC SacI fwd: 5’-GAGCTCGGACTTTTGCG-3’</p><p>GC KpnI rev: 5’-GGTACCTTTTTTTTTTTTTTTTT-3’</p><p>After consulting Sebastian Palluk, who worked with this kind of PCR before, we decided to design longer primers, with different lengths, so that they have a similar melting temperature and a high annealing temperature. We designed these primers for our “Normal Primer”. They have an overhang with the same restriction cut sites as well, but we added a few bases, which should enable a better hold for the restriction enzymes.</p><p>N_fwd: 5’-TTTTGAGCTC ACAATTCATCTTAGTGCCAGACTAC-3’</p><p>N_rev: 5’-TTTTGGTACCTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT-3’</p><h2>Short Oligos for Hybridization</h2><p>We also designed short oligos with a ddNTP at the 3’-end to stabilize the single-strand during elongation, but we could not properly test them in the lab.</p><p>3A3C: 5’-AAACC/ddC/-3’</p><p>5A5C: 5’-AAAAACCCC/ddC/-3’</p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Lee, H., Wiegand, D. J., Griswold, K., Punthambaker, S., Chun, H., Kohman, R. E., &amp; Church, G. M. (2020).</p><cite>Photon-directed multiplexed enzymatic DNA synthesis for molecular digital data storage.</cite><p><span class="journalTitle">Nature Communications</span> <span class="journalInfo">11(1).</span></p><a class="in-text" href="https://doi.org/10.1038/s41467-020-18681-5" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Photon-directed multiplexed enzymatic DNA synthesis for molecular digital data storage." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author"></p><cite>DNA Fragment Analysis. Fraunhofer IME.</cite><p><span class="details">Retrieved on October 08, 2021. from </span><a class="in-text" href="https://www.ime.fraunhofer.de/en/Research_Divisions/business_fields_MB/services/dna_fragmentanalyse.html" target="_blank" rel="noopener">https://www.ime.fraunhofer.de/en/Research_Divisions/business_fields_MB/services/dna_fragmentanalyse.html</a></p><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author"></p><cite>PCR Troubleshooting - Causes Related to Cycling Times and Temperatures. BIO-RAD.</cite><p><span class="details">Retrieved on October 08, 2021. from </span><a class="in-text" href="https://www.bio-rad.com/de-de/applications-technologies/pcr-troubleshooting?ID=LUSO3HC4S#:~:text=Annealing%20temperature%20was%20too%20low,Tm%20of%20the%20primer" target="_blank" rel="noopener">https://www.bio-rad.com/de-de/applications-technologies/pcr-troubleshooting?ID=LUSO3HC4S#:~:text=Annealing%20temperature%20was%20too%20low,Tm%20of%20the%20primer</a></p><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Sahlin, K., &amp; Medvedev, P. (2021).</p><cite>Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis.</cite><p><span class="journalTitle">Nature Communications</span> <span class="journalInfo">12(1).</span></p><a class="in-text" href="https://doi.org/10.1038/s41467-020-20340-8" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Error correction enables use of Oxford Nanopore technology for reference-free transcriptome analysis." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Zhang, Y., Zhou, H., &amp; Ou-Yang, Z.-C. (2001).</p><cite>Stretching Single-Stranded DNA: Interplay of Electrostatic, Base-Pairing, and Base-Pair Stacking Interactions.</cite><p><span class="journalTitle">Biophysical Journal</span> <span class="journalInfo">81(2), 1133-1143.</span></p><a class="in-text" href="https://doi.org/10.1016/s0006-3495(01)75770-0" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Stretching Single-Stranded DNA: Interplay of Electrostatic, Base-Pairing, and Base-Pair Stacking Interactions." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Delarue, M., Boul , J. B., Lescar, J., Expert-Bezan on, N., Jourdan, N., Sukumar, N., ... &amp; Papanicolaou, C. (2002).</p><cite>Crystal structures of a template-independent DNA polymerase: murine terminal deoxynucleotidyltransferase.</cite><p><span class="journalTitle">The EMBO Journal</span> <span class="journalInfo">21(3), 427-439.</span></p><a class="in-text" href="https://doi.org/10.1093/emboj/21.3.427" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Crystal structures of a template-independent DNA polymerase: murine terminal deoxynucleotidyltransferase." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Shishmarev, D., Wang, Y., Mason, C. E., Su, X.-C., Oakley, A. J., Graham, B., ... &amp; Otting, G. (2013).</p><cite>Intramolecular binding mode of the C-terminus of Escherichia coli single-stranded DNA binding protein determined by nuclear magnetic resonance spectroscopy.</cite><p><span class="journalTitle">Nucleic Acids Research</span> <span class="journalInfo">42(4), 2750-2757.</span></p><a class="in-text" href="https://doi.org/10.1093/nar/gkt1238" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Intramolecular binding mode of the C-terminus of Escherichia coli single-stranded DNA binding protein determined by nuclear magnetic resonance spectroscopy." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="../assets/img/all_sponsors.png"></div></div></footer><script src="../content.bundle.js"></script></body></html>