<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="utf-8"/><meta content="width=device-width,initial-scale=1" name="viewport"/><title>Notebook | iGEM Aachen</title><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="https://2021.igem.org/Template:Aachen/css/contentCSS?action=raw&amp;ctype=text/css" rel="stylesheet"/></head><body data-offset="100" data-spy="scroll" data-target="#contentnav"><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://2021.igem.org/Team:Aachen"><img src="https://2021.igem.org/wiki/images/1/14/T--Aachen--img--teamlogo_2021.svg"/></a><button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown" role="button">Project</a><div aria-labelledby="navbarProjectDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Description">Description</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Implementation">Implementation</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Engineering">Engineering</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Contribution">Contribution</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Memes">Memes</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarWetlabDropdown" role="button">Wetlab</a><div aria-labelledby="navbarWetlabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Experiments">Experiments</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Notebook">Notebook</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Protocols">Protocols</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Results">Results</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Outlook">Outlook</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Safety">Safety</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarDrylabDropdown" role="button">Drylab</a><div aria-labelledby="navbarDrylabDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Hardware">Hardware</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Model">Modeling</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Software">Software</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarHuman PracticesDropdown" role="button">Human Practices</a><div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Science_Communication">Science Communication</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Collaborations">Collaborations</a></div></li><li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true" class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarTeamDropdown" role="button">Team</a><div aria-labelledby="navbarTeamDropdown" class="dropdown-menu"><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Members">Members</a><a class="dropdown-item" href="https://2021.igem.org/Team:Aachen/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="https://2021.igem.org/wiki/images/8/88/T--Aachen--img--headers--Notebook.jpg"/><img class="header-text" src="https://2021.igem.org/wiki/images/f/fe/T--Aachen--img--headers--header_texts--Notebook.svg"/><img class="header-effect" src="https://2021.igem.org/wiki/images/d/d7/T--Aachen--img--headers--header_effect.svg"/></header><nav id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>TdT Tailing Reaction</h1><h1>Primer Immobilization</h1><h1>Cyclic synthesis</h1><h1>Secondary structures</h1><h1>Week 16</h1><p>In the beginning, we played around with the TdT tailing reaction conditions to get accustomed to the enzyme and figure out ideal settings.<br/>Our first tests with the TdT from NEB were simple elongation reactions with an accidentally very low concentration of a short primer and individual dNTPs in each case. In the experiment, we also compared incubation at room temperature and 37 °C. The reactions were stopped by heat inactivation after different periods and analyzed by 2% agarose gel electrophoresis using Roti GelSTain Red.<br/>Additionally, we measured the nucleic acid concentrations in our samples with the nanodrop.</p><h1>Week 17</h1><p>Week 17 started with shorter TdT tailing reactions using only dATP and dTTP. After heat inactivation, we combined samples with different nucleotides and performed a PCR. The nucleic acid concentrations were measured before and after the amplification and the samples were analyzed via agarose gel electrophoresis.<br/>The dATP/dTTP tailing reactions were repeated at a higher incubation temperature, longer incubation time and with a lower primer concentration. To compare the strain lengths and to visualize different primer concentrations we analyzed various dilutions of our short primers and the samples produced during the week on 2% and 2.5% agarose gels. As usual, we measured the nucleic acid concentration with the nanodrop. Additionally, we repeated the dATP and dTTP TdT reactions with the changed reaction settings once more.</p><h1>Week 18</h1><p>As a new analyzing method we performed native polyacrylamide (PAA) gel electrophoresis on the dATP/dTTP TdT tailing reactions from week 17 together with different primer concentrations. To visualize our results, we tried working with silver staining instead of the usual fluorescent dyes. Furthermore, to determine the detection limits in agarose gel electrophoresis we diluted new primers and compared different concentrations on agarose, native and denaturing PAA gels also using silver staining on the latter.</p><h1>Week 19</h1><p>Switching to a new fluorescent stain (SYBR Gold) and a higher resolution DNA ladder (GeneRuler Ultra Low Range), we continued to analyze primer dilutions on agarose gels. We also altered the agarose concentration (1-3%) in several experiments to evaluate the consequences.<br/>The new staining agent was also tested on native and denatured PAA gels. After determining ideal agarose gel conditions new TdT tailing reactions with different primer dilutions were performed. The reactions were stopped by inactivation of the TdT via EDTA and the samples analyzed with agarose gel electrophoresis. Because there were still some issues with the PAA electrophoreses, we tested different ladder and primer dilutions. To test the new type of DNA ladder we also ran an agarose gel electrophoresis with a 5% of agarose.</p><h1>Week 20</h1><p>Week 20 started with TdT tailing reactions, which were inactivated with EDTA after different periods. The length of the synthesized fragments was analyzed on native PAA and agarose gels. To characterize another aspect of the TdT enzyme, the following reactions were done with different primer dilutions. We also compared the inactivation via EDTA with heat inactivation in the same experiment. Analysis was done repeatedly with native and denaturing PAA and agarose gels on two consecutive days. Trying to optimize the shape of the electrophoresis bands we mixed our gel samples (including diluted samples of ladder and primer) with glycerol before applying them on native or denaturing PAA and 5% agarose gels.<br/>Still using the TdT tailing samples from the beginning of the week, we analyzed them further with longer staining times and more loading dye. The next TdT tailing reactions were performed with different nucleotides in each TdT reaction, also comparing 15 and 40 minutes incubation time. Analysis was done with agarose gel electrophoresis.</p><h1>Week 21</h1><p>We continued to work on the gel electrophoresis band shape issue while trying out pre-running PAA gels at a lower voltage for a few minutes before the actual electrophoresis. Also trying to improve the band separation, agarose gel electrophoresis was performed on a primer dilution and a TdT sample with 5% agarose, higher voltages and shorter run times. Addressing this we compared the TdT samples from last week on agarose gels made with TBE to gels with TAE (as usual).</p><h1>Week 22</h1><p>Analyzing the nucleotide preferences of the TdT, we performed several TdT tailing reactions with one type of nucleotide each and a different short primer. Due to several conspicuities during the analysis, we ran multiple agarose gel electrophoreses.<br/>Switching to an AT-rich 100 bp primer we started using another ladder (GeneRuler 50bp DNA ladder) and 2.5% agarose gels for analysis.<br/>We then continued to perform TdT tailing reactions with different nucleotides each and the new longer primer.<br/>Furthermore, we compared the performance of the TdT using 20 bp primer with the 100 bp primer reactions.</p><h1>Week 23</h1><p>We recreated the successful reactions from last week with the 100 bp primer using the Thermofisher TdT analyzing its nucleotide preferences. We also modified the primer concentration in the tailing reactions. After the gel electrophoresis, we used an ssDNA Recovery Kit to purify strands that were elongated with only dATP or dTTP from the gel. To check the reproducibility of the primer comparing experiment, we did the same experiment twice at the beginning of the week. Further analyzing the nucleotide preferences of the TdT and the formation of secondary structures we performed standard TdT tailing reactions with a 100 bp GC-rich primer. We also altered the primer concentration (AT-rich) again and tested, what effect on the strand elongation a further dilution of the primer has. Addressing another aspect of the TdT tailing reactions we performed experiments with different incubation times from 1-60 minutes using only dATP and the AT-rich primer.<br/>We repeated the experiment, using two different dilutions of the primer (100nM as before and 20nM) and instead of incubating the different reaction set-ups individually as the day before we withdrew a small amount of sample at the corresponding times (1-120 minutes). Two samples were also left in a 37 °C room for overnight incubation and analyzed on a gel the next day compared to 120-minute-long incubated samples. After that, we purified the overnight samples with a DNA recovery kit and measured the nucleic acid concentrations with nanodrop.<br/>In a new TdT tailing reaction, we used the purified samples as primers and elongated the strands with dATP again.</p><h1>Week 24</h1><p>We performed - again - a TdT tailing reaction with the NEB TdT and changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 30 C° and the incubation times. After that, we isolated DNA from the produced gel to recover elongated ssDNA strands and measured their DNA concentration via the Nanodrop.<br/>Moreover, we repeated the TdT tailing reaction with the ThermoFisher TdT and also changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 37 C° and the incubation times.<br/>Subsequently, we repeated the TdT tailing reaction with the ThermoFisher TdT again because we used the wrong buffer volume the day before.<br/>In another experiment, we investigated the TdT tailing reaction of samples that either contain or do not contain CoCl2 at room temperature and 37 °C, to see if the elongation occurred independently of the co-factor.<br/>Then, we prepared a dNTP dilution of 100 µM and performed a TdT tailing reaction with the ThermoFisher TdT to compare the impact of the usage of dATP, dTTP, dGTP and dCTP on the reaction.</p><h1>Week 25</h1><p>To get a standard gel reference for the ThermoFisher TdT we performed a TdT reaction containing a single type of dNTP. We repeated this experiment because it did not work the first time.<br/>In another experiment, we tested different incubation times with dATP at room temperature and 37 °C to investigate the optimal TdT reaction conditions.<br/>We also repeated the TdT tailing reaction with the ThermoFisher TdT. This time the TdT reactions contained a single type of dNTP incubated at room temperature and 37 °C for 30 min to get a standard gel reference for the ThermoFisher TdT.<br/>Then we performed a time-dependent TdT reaction, where we tested different incubation times to estimate the speed at which nucleotides were added.<br/>Additionally, a TdT tailing reaction with different buffer concentrations was performed to test the impact on elongation.<br/>We also performed a two-step TdT reaction, where we did a TdT tailing reaction, inactivated it, purified a part of the ssDNA and performed another TdT tailing reaction with the purified DNA to verify its impact on the elongation.<br/>Then we performed a TdT tailing reaction with different TdT concentrations and dGTP to investigate if it creates sharper bands.</p><p>Finally, we performed a vector digestion for cloning with a pMA_217 vector and the enzymes Kpnl and Sacl.</p><h1>Week 26</h1><p>In this week we performed a TdT tailing reaction with different nucleotides and the fluorescent primer GC-rich for capillar electrophoresis (CE). Additionally, we repeated this experiment with the fluorescent primer AT-rich. Then, different concentrations of the fluorescent primers were tested for a better detection of the TdT tailing reaction. Also, we compared the Roti GelStain Red and SYBR Gold staining as well as the DNA and RNA Ladder.<br/>Furthermore, the primers and compiled samples were analyzed with CE.</p><p>Then we tested the TdT reaction with the fluorescent primer GC-rich and SSBs to detect whether our SSBs prevent secondary structures in a reaction and/or resolve them.<br/>Moreover, we tested the TdT reaction with the GC-rich primer and different SSBs concentrations to detect whether our SSBs prevent secondary structures and to investigate the effect of SSBs on the elongation.</p><p>We also started experiments with the immobilization system by testing the binding of fusion proteins to a polystyrene stick. For this purpose, we used the anchor peptides LCI and MacHis and performed an ELISA immunoassay.</p><h1>Week 27</h1><p>In this week we reanalyzed the samples from week 26 with the CE.<br/>We also repeated the digestion of the vector pMA_217 for the cloning of our insert, because it did not work the first time.<br/>Furthermore, we performed a TdT tailing reaction with the ThermoFisher TdT and dATP as well as with dATP, dCTP and dGTP mixed at room temperature and incubated for 30 min to get two inserts for cloning. With the obtained samples, a PCR was performed to get our inserts for the cloning reaction. We repeated the PCR reaction because it did not work the first time.<br/>We also diluted the HPLC purified fluorescent primers.<br/>Then we performed a PCR reaction with the generated PCR reaction samples to increase the concentration and to get our inserts for the cloning reaction.<br/>Subsequently, we prepared new samples for the CE. This time we used the HPLC purified primers and purified the samples before analyzing them by the CE. We also used short reaction times ranging from 5-120 s with every nucleotide to determine the reaction speed for each nucleotide.<br/>Finally, we analyzed the samples with the CE.</p><h1>Week 28</h1><p>To get a better understanding of how the ThermoFisher TdT works, we tested different nucleotide concentrations because they can increase or slow down the TdT reaction speed.</p><p>We also performed another test run with a range of SSB:Primer ratios to test the effect of SSBs on the ThermoFisher TdT reaction.</p><p>Furthermore, we performed a seamless cloning reaction (SCL).<br/>Therefore, we did a transformation with <em>E. coli</em> C3040I chemically competent cells, the pMA_217 vector and our poly-A and ACG inserts.<br/>Subsequently, we did a colony PCR to verify if the inserts were incorporated successfully into <em>E. coli</em> C3040I.<br/>Then we purified the samples ACG5, A14, A20 and A24, measured their concentrations and prepared them for sequencing with the Eurofins-Mix2Seq-Kit.<br/>Afterward, we analyzed the sequencing data by aligning the sequences to verify if there were just re-ligands or if the inserts were incorporated into the vector.<br/>Moreover, we did a Nanopore sequencing with the MinION and the help of Max Schmidt. Here, we sequenced the PCR samples of the poly-A insert and the ACG insert.</p><h1>Week 29</h1><p>We tested new TdT tailing conditions with the ThermoFisher TdT. This time we used different cofactor (CoCl2) concentrations without the reaction buffer.</p><p>Moreover, we repeated the SSB experiment from last week with one extra negative control and did a PCR reaction with these samples. Subsequently, we tried to denaturate the SSBs properly by heating the samples up to 95 °C, because the SSBs were still binding to the ssDNA.</p><p>We also tested the conjugation of a AT_Oligo_C6_linker primer with an amino C6-linker protein to establish our immobilization system.<br/>Since the immobilization of the primer did not work, we wanted to create a new reaction in solution to test the immobilization system.</p><h1>Week 30</h1><ul><li></li></ul><h1>Week 31</h1><p>We repeated the SSB experiment of week 29 and the subsequent PCR.</p><p>Furthermore, we repeated the immobilization experiment of week 29 to test the TdT reaction with the immobilized primer and with the primer-protein conjugate. We wanted to verify if an elongation took place with a primer-protein conjugate and to test if we had some conjugate that sticks to the polystyrene stick and if we could elongate them.<br/>We also were not sure if the immobilization did not work or just the conjugation or both, so we performed an SDS PAA gel to detect the peptide of the samples from the AT-rich C6 stick, the stick with the peptide and the primer, and the stick with the peptide and primer reaction.<br/>Furthermore, we repeated the immobilization with the conjugate and analyzed it with an agarose gel electrophoresis.</p><p>We also prepared samples for a pGEM cloning by performing a TdT tailing reaction with dATP as well as dATP, dCTP and dGTP to get two inserts for cloning. Subsequently, we performed a PCR with the samples to get our inserts for the pGEM cloning.</p><h1>Week 32</h1><p>In this week, we used the pGEM-T Easy Vector System for cloning and performed a ligation and a transformation in <em>E. coli</em> C3040I.<br/>Then, we performed a colony PCR to verify if the inserts were incorporated successfully.<br/>Subsequently, we prepared samples for the sequencing with the Eurofins-Mix2Seq-Kit and finally analyzed the sequencing data to verify if the inserts were incorporated into the vector.<br/>We repeated the ligation and transformation experiment with the pGEM-T Easy Vector System because it did not work the first time.</p><p>Furthermore, we performed a conjugation of a maleimide-primer with peptide. Our aim was to link the primer with the anchor peptide to establish our immobilization system.</p><h1>Week 33</h1><p>In the beginning, we performed a TdT tailing reaction with dATP and a mixture of ACG nucleotides to amplify our samples with a PCR reaction and get two inserts for the pGEM cloning reaction. We generated the poly-A tail ourselves, by adding 1 µL of nucleotide A to our reaction. At first, it did not seem to work because we saw no difference in the length distributions of our bands. Despite that, we carried on to perform the PCR reaction and saw a blurry elongation for the ACG reactions.</p><p>We continued by using a DNA recovery kit and measured our dissolved DNA with the Nanodrop to get an idea of the final concentrations. Furthermore, we analyzed last week's primer–anchor peptide conjugates with an SDS Page and Western blot to examine the protein. Afterward, we prepared Maleimid Primer – peptide reactions to assess the conjugation rate further.</p><h1>Week 34</h1><p>This week we first analyzed a TdT reaction with conjugated Primers in solution. We decided to keep the stock concentration of the protein constant. Therefore the resulting concentration of the primer in the TdT tailing reaction varied.</p><p>The next day we began to purify the protein-primer-conjugate by using nickel beads to analyze the concentrations and show whether the conjugates would be visible on the protein gel. Later we analyzed both the purified samples and the flow-through on an agarose gel to see the effect on the purification and analyze the success of washing and purification. Unfortunately, we estimated the primer and protein concentration wrong, so the references were not visible. Thus we repeated the experiment and re-analyzed the TdT samples in higher concentrations. After the successful experiment yesterday, we carried out our first immobilization with the maleimid-conjugates on a stick. We assessed different reaction conditions, wash steps, and negative controls to gather as much information from of the samples.</p><h1>Week 35</h1><p>The next day, new TdT reactions were analyzed with Thermo TdT and NEB buffer to check if the immobilizations worked better. To increase the concentration of the primer-protein conjugate, we first started a TdT reaction and immobilized the primer during the reaction.<br/>Continuing our first experiments, we focused on analyzing whether a higher primer concentration influenced the running characteristics of the agarose gel. Therefore we created TdT reactions with primer concentrations from 20 to 1000 nM for the AT-rich as well as the GC-rich primer and analyzed them with an agarose gel.</p><p>This week we focused on continuing to examine the Maleimid- immobilization on a stick. We repeated the experiment from last week with LCI-GC 1 on a smaller scale and tested the influence of the buffer on the adsorption of the anchor peptide. No bright bands were visible so we assessed the influence of the buffer again. To obtain clear immobilization results, we wanted to examine wheater Triton-X-100 (a component of the NEB TdT reaction buffer) could serve as a potential alternative to the ThermoFisher TdT reaction buffer. Thus we prepared LCI GC 1 samples with different buffer compositions. Again, no bands were visible. After yesterday's failed experiment, we attempted to measure the conjugate samples with the nanodrop to obtain a greater insight into the „real“ concentrations of the conjugates. We measured 1 µL of all samples and gained significantly lower results than expected. Furthermore, we evaluated the influence of the buffer composition on the TdT reaction again with a different setup. This time, the samples were prepared with LCI AT and one µM conjugate concentrations (much higher than before).<br/>Samples with AT-Maleimid Primer were prepared as well to function as a negative control to see if the primer would stick on its own. As a result, faint bands were visible. Additionally, the samples were measured with the nanodrop to gather additional information.</p><h1>Week 36</h1><p>Optimizing our PCR reactions, different amounts of template DNA were used. In order to make this possible, we prepared 5 PCR mixes with different volumes of template DNA using the nucleotide T and reference reactions. The results were not as expected. Consequently, the samples were applied again with normal TdT reactions as reference. Following up on the previous experiment, the PCR reaction with the GC primer was repeated with an increased annealing temperature. Unfortunately, the wrong primers were used for the PCR reaction, so the experiment had to be repeated.</p><p>Since the majority of the results from last week were inconclusive, we continued experimenting with the NEB buffer and TdT reactions with different primer concentrations. In the first experiment, we used the LCI-GC1 conjugate and mixed NEB and ThermoFisher TdT/buffer. Because the results were too vague, the experiment was repeated with the same samples. In week 34 high bands were obtained, which seemed to be stuck in the pockets of the gel. To analyze whether the TdT can elongate this far, different primer concentrations were examined with a normal TdT reaction. After a reaction time of 25 mins, an agarose gel was obtained. The remaining samples were not inactivated and the reaction was incubated overnight. The electrophoresis was inconclusive, so a new gel was obtained. The next day, the overnight incubated samples were applied on a gel to see how far the reaction was elongated. Furthermore, a PCR reaction with samples from the immobilization experiment from KW 34 was driven. All conjugates were analyzed and a gel with positive, negative controls, Primer-Conjugate immobilization reactions, and their PCR reactions was acquired. To see if the visible bands were not-conjugated primers or parted conjugate from the stick, the bands from the gel were excised and analyzed. The NucleoSpin-PCR-cleanup-kit was used according to the protocol. The resulting samples were transferred onto a Trans-Blot Turbo membrane and the visible spots were assessed. Additionally, the immobilization on cuvette sticks with direct application of SYBR gold was carried out to show the advantage of the immobilized conjugate in contrast to the pure primer. Thus, drying steps between the immobilization and washing were incorporated as well as applying small amounts of primer and conjugate directly to the stick. Comparing the ThermoFisher and NEB TdT reaction buffer, TdT-maleimid-Immobilisation reactions were carried out with LCI-AT conjugates and negative controls with AT-rich primers. To further assess the reaction conditions of the NEB buffer, different incubations times from 30 minutes up to 230 mins were tested and for GC-rich and AT-rich primer compared.</p><h1>Week 37</h1><p>Continuing our proof of concept work, we finally created a simple TdT reaction with several base transitions. Two reactions were performed in parallel, one with AT-rich primers and one with CE-labelled AT-rich primers for a potential analysis with the capillary electrophoresis. The nucleotides were added in the following order: T, C, G, A with each 10 min incubation time. Nucleotide A was added last to create a Poly-A tail to allow a later PCR amplification. The experiment worked and bands were visible on the gel.</p><p>As the results from the last immobilization were flawed, the experiment was repeated with a few modifications. The stick surface was treated with sandpaper to create a rough surface, the initial dNTP concentration was increased, and the mobilization was performed with Triton-X-100 instead of heat. The Primer-immobilisation on cuvette stirrers was performed according to protocol.</p><p>This week we started our first tests with small oligos (from now on called SO's), which hybridized with the elongated primers, to analyze the prevention of secondary structures. TdT reaction with nucleotides T and G and the GC-rich primer were carried out. The small oligos were added to the TdT reaction mix and different concentrations were tested. The results were inconclusive.</p><h1>Week 38</h1><p>To recreate a cyclic TdT reaction, the experiment from last week was repeated and a few changes were performed. The incubation time was changed from 10 to 15 minutes and the dNTP reaction was significantly increased to obtain clearer results hopefully.</p><p>Furthermore, we continued the Hybridisation experiment from last week by re-running the samples without the 3A3C oligo while testing the 5A5C oligo. Unfortunately, the experiment had to be repeated because the normal TdT reaction did not show a band on the gel.<br/>Additionally, we aimed to test which SO concentration worked best for our 3A3C sample, so we examined different volumes of SO's in our reactions ( 1 to 20 µL ).<br/>Continuing the previous experiment, we repeated the reaction conditions with the 5A5C SO's and the volumes 1 µL and 10 µL.</p><h1>Week 39</h1><p>Because our last PCR reactions were inconclusive we drove a gradient PCR to determine the best annealing temperature. For this experiment, TdT reactions with different primers and different nucleotides were carried out to test wheater secondary structures have an impact on the PCR.<br/>As the samples were smeared and had bad visibility on the gel, new samples with only one nucleotide were created.<br/>After the electrophoresis, the results were not as expected, so another gradient PCR was driven to determine the best annealing temperature for our AT-rich and GC-rich primer. We used the Thermo Fisher calculator for the annealing temperature and experimented with a temperature range from 35 – 55 °C. Additionally, we repeated the experiment from 14.09 (Week 37) to create simple cyclic TdT reactions with base transitions. A few changes occurred: We used less primer (100 nM instead of 500 nM in comparison to 14.09), incubated at a higher temperature, and used longer incubation times (15 to 20 mins instead of 10 mins). Half of the samples were saved and put in the fridge for analysis next week.</p><p>This week we started our first tests with streptavidin-tagged magnetic beads and biotin-labeled primers to establish our Biotin-Streptavidin Immobilisation system as an alternative to the Maleimid-immobilization. We performed one reaction and two negative controls without biotin and without beads to check the turnover rate of our immobilization. Unfortunately, we ordered the primer with the biotin-group on the wrong end so the Experiment failed.</p><h1>Week 40</h1><p>Since last week's gradient PCR samples were smeared, we performed a PCR reaction with the high smear that we isolated from the gel to hopefully amplify samples with the desired length. Afterward, we performed a PCR reaction with the samples from 30.09 to prepare them for sequencing and check for the desired bands. The PCR products were cast on an agarose gel. Additionally, we prepared TdT reactions with the „N-Primer“ to create a better analysis method for PCR reactions. We started doing standard TdT samples with different incubations times for each nucleotide and added a Poly-A tail to half the samples to use for a later PCR reaction. The results were inconclusive.</p></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="https://2021.igem.org/wiki/images/a/af/T--Aachen--img--all_sponsors.png"/></div></div></footer><script src="https://2021.igem.org/Template:Aachen/content-bundleJS?action=raw&amp;ctype=text/javascript"></script></body></html>