<html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Proof of Concept | iGEM Aachen</title><script type="text/x-mathjax-config" async>MathJax.Hub.Config({
  tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
});</script><script src="https://2021.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" async></script><script>function resizeSidebar() {document.getElementById("contents").style.height = (document.getElementById("list").scrollHeight + 100 + "px"); }
window.onresize = resizeSidebar;</script><link href="../css/content.css" rel="stylesheet"></head><body data-spy="scroll" data-target="#contentnav" data-offset="150" onload="resizeSidebar()"><nav class="navbar navbar-expand-xl fixed-top"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="/"><img src="../assets/img/teamlogo_2021.svg"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="/Description">Description</a><a class="dropdown-item" href="/Implementation">Implementation</a><a class="dropdown-item" href="/Engineering">Engineering Success</a><a class="dropdown-item" href="/Proof_Of_Concept">Proof of Concept</a><a class="dropdown-item" href="/Contribution">Contribution</a><a class="dropdown-item" href="/Outlook">Outlook</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarWetlabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Wetlab</a><div class="dropdown-menu" aria-labelledby="navbarWetlabDropdown"><a class="dropdown-item" href="/Experiments">Experiments</a><a class="dropdown-item" href="/Notebook">Notebook</a><a class="dropdown-item" href="/Protocols">Protocols</a><a class="dropdown-item" href="/Results">Results</a><a class="dropdown-item" href="/Safety">Safety</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarDrylabDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Drylab</a><div class="dropdown-menu" aria-labelledby="navbarDrylabDropdown"><a class="dropdown-item" href="/Hardware">Hardware</a><a class="dropdown-item" href="/Model">Modeling</a><a class="dropdown-item" href="/Software">Software</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman PracticesDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices</a><div class="dropdown-menu" aria-labelledby="navbarHuman PracticesDropdown"><a class="dropdown-item" href="/Human_Practices">Human Practices</a><a class="dropdown-item" href="/Communication">Science Communication</a><a class="dropdown-item" href="/Collaborations">Collaborations</a><a class="dropdown-item" href="/Memes">Memes</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="/Members">Members</a><a class="dropdown-item" href="/Attributions">Attributions</a></div></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><img class="header-image" src="../assets/img/headers/Proof of Concept.jpg"><img class="header-text" src="../assets/img/headers/header_texts/Proof of Concept.svg"><img class="header-effect" src="../assets/img/headers/header_effect.svg"></header><nav class="bg-image" id="contentnav"><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents" style="flex-direction: column"><h5>Proof of Concept</h5><ul id="list"></ul></div></div><div class="content col-lg-9"><article><h1>Overview</h1><p>Since the official start of the competition in April 2021, we were able to develop a method to synthesize ssDNA strands with a pre-set sequence. Our project covered many different aspects like the immobilization of DNA, synthesis with our <a href="https://2021.igem.org/Team:Aachen/Hardware">Hardware</a> and experimental confirmation through <a href="https://2021.igem.org/Team:Aachen/Model">Modeling</a> as well as encoding and decoding information in DNA with our <a href="https://2021.igem.org/Team:Aachen/Software">Software</a>. We want to demonstrate that these parts can work together, thus making our project feasible as a whole. Inspired by the scientific findings of George Church <a href="#citation1">1</a>, we would like to present a concept that is simple and efficient at the same time.</p><h1>DNA Synthesis with TdT</h1><p>The performance of an optimal reaction in the later course of our project required a precise characterization of our enzyme, see <a href="https://2021.igem.org/Team:Aachen/Results">Results</a>. For basic research on TdT, the common co-factor $Co^{2+}$ was used. We investigated which nucleotides were preferentially incorporated, how far the initial strand could be elongated with a nucleotide species, and how temperature affects elongation. The evaluation showed that in a 10 min reaction, strands with the nucleotides thymine, cytosine, and adenine are elongated better than strands with the nucleotide guanine. Temperature also significantly affected elongation; the strands get longer at 37 °C than at RT (figure 1). To be able to investigate whether all strands were elongated equally in a reaction, capillary electrophoresis was applied to individual approaches. It was found that there were individual differences between the strands. The deviation was in the range of up to 30 nt. We decided on the lower incubation temperature at RT because it is easier to implement and our goal is to attach only a few nucleotides. A final important point to implement the most uniform elongation possible in the further course is the primer:nucleotide ratio in the solution. At a ratio of 1:5000, strands were elongated uniformly to an approximate length of up to 1000 nt (see figure 1 lane 8). If the ratio of primer to nucleotide was 1:500, very few nucleotides were incorporated (maximum up to 50). This data was evaluated further in our model.</p><div class="image"><img src="../assets/img/Results/2021-06-23_01.png" alt="&lt;b&gt;TdT tailing reaction of an AT-rich ssDNA primer with the four standard dNTPs at RT and 37 °C for 30 min, containing $Co^{2+}$ as a cofactor.&lt;/b&gt; 2.5% agarose with TBE, SYBR Gold staining, 90 V, 55 min. M: GeneRuler 50 bp DNA Ladder 1: AT-rich primer reference 2: RT reaction with dATP 3: RT reaction with dTTP 4: RT reaction with dCTP 5: RT reaction with dGTP 6: AT-rich primer reference 20 nM 7: 37 °C reaction with dATP 8: 37 °C with dTTP 9: 37 °C reaction with dCTP 10: 37 °C reaction with dGTP." style="width: 100%"><p style="font-size:0.8rem; text-align:justify;"><b>Figure 1</b>: <b>TdT tailing reaction of an AT-rich ssDNA primer with the four standard dNTPs at RT and 37 °C for 30 min, containing $Co^{2+}$ as a cofactor.</b> 2.5% agarose with TBE, SYBR Gold staining, 90 V, 55 min. M: GeneRuler 50 bp DNA Ladder 1: AT-rich primer reference 2: RT reaction with dATP 3: RT reaction with dTTP 4: RT reaction with dCTP 5: RT reaction with dGTP 6: AT-rich primer reference 20 nM 7: 37 °C reaction with dATP 8: 37 °C with dTTP 9: 37 °C reaction with dCTP 10: 37 °C reaction with dGTP.</p></div><h1>Modeling Experimental Data</h1><p>When comparing the strand length distribution calculated by our <a href="https://2021.igem.org/Team:Aachen/Model">Model</a> and the one determined by capillary electrophoresis one can see that the model is relatively accurate. Even though the numbers are not exactly the same, the general distribution properties were very similar. The exact numbers however could be used to determine new kinetic constants and build a more accurate model, which corresponded even more to the experimental data.</p><p>Concerning the primer:nucleotide ratio, our modeling shows that it should make no difference if we incubate a longer time with a low nucleotide concentration or a shorter time with a high nucleotide concentration. As shorter incubation times are more favorable for a future synthesis system because time becomes an essential factor in large-scale production, we decided on the high nucleotide concentration for our immobilization experiments. This indicates that a combination of high primer:nucleotide ratio with shorter incubation time and lower temperature would be crucial for our further experiments.</p><h1>DNA Immobilization</h1><p>For the immobilization of our ssDNA primer on magnetic beads, we decided to use the streptavidin-biotin-bond (see <a href="https://2021.igem.org/Team:Aachen/Experiments#CyclicSynthesisandPrimerImmobilization">Experiments</a>), as this is one of the strongest non-covalent interactions <a href="#citation2">2</a>. A neodymium magnet stick was used to extract the beads, no loss of primer could be detected in the conjugation or washing steps (see <a href="https://2021.igem.org/Team:Aachen/Results#2PrimerImmobilization">Results</a>). Thus, there was a low risk of contamination between the reaction solutions. Using a 100 nt primer prevented steric hindrance.</p><p>The magnets were equipped with a pipette tip as an envelope (the bottom opening of the tip was sealed) to prevent permanent binding of the beads to the magnet and to be able to reuse the materials.</p><p>We showed that the ssDNA primers can be extracted from the solution via magnetic beads and a magnet stick and that the DNA can be eluted with no significant loss after several washing steps. It was shown that elongation is still possible in the immobilized state. Furthermore, the envelope for the magnet allowed us to release the beads into the reaction solutions, which resulted in better elongation.</p><h1>Hardware Design</h1><p>To automate our method, we developed a custom design, see <a href="https://2021.igem.org/Team:Aachen/Hardware">Hardware</a>. The individual reactions take place in 2 mL Eppendorf reaction tubes. In order to automate the elongation and washing steps, a framework of extruded aluminum profiles and a revolver with a mobile base was designed. The revolver is used to fix the reaction vessels. The stepper motor, which rotates the revolver, enables a maximum accuracy of 0.1125° through microstepping. A rod clamp can be attached to the plate to fix the rod magnet. The plate can be moved vertically, which allows the magnet to be immersed in the solution. This approach was chosen because it is simple, efficient, and feasible. The process is automated by a NodeMCU, which provides a simple web interface where the machine can be calibrated with one click of a button. In the next step, only the DNA sequence has to be specified, and the synthesis runs without further manual steps.</p><h1>Data Storage in DNA</h1><h2>The DIP Method</h2><p>In the DIP method, an immobilized ssDNA primer is dipped into a reaction solution containing the enzyme TdT and one type of nucleotide (see <a href="https://2021.igem.org/Team:Aachen/Experiments#CyclicSynthesisandPrimerImmobilization">Experiments</a>). The goal is to generate a strand as long as possible with the maximum number of transitions from different nucleotides. Since the ssDNA strands had to be double-stranded for subsequent analysis (sequencing using Sanger and Nanopore), performing a PCR is essential. To make this possible, a poly-A tail has to be present at the end of each strand for the binding of a reverse primer, so the addition of dATP is necessary at the end of the reaction.</p><h2>Retrieval of Data</h2><p>After successfully immobilizing a primer to a magnet stick and proving that the TdT is able to attach different kinds of nucleotides to an immobilized primer, we were able to put our DIP method to use. This enabled encoding data into DNA by creating up to five transitions and reading these with an accuracy of 80% (see <a href="https://2021.igem.org/Team:Aachen/Results#CyclicSynthesis-Immobilized">Results</a>). The fact that we were able to achieve such a high accuracy with the first attempts indicates the high potential of this method. This was made possible by the DIP method invented by us in combination with our self-developed software <a href="https://2021.igem.org/Team:Aachen/Software">DNA Utils</a>. DNA Utils is able to encode arbitrary files into DNA sequences and decode these sequences to retrieve the original files. But most importantly, we showed that it is able to transform actual sequencing data back into the original sequence which then can be decoded.</p></article><div class="row justify-content-center"><a class="tourButton" href="https://2021.igem.org/Team:Aachen/Outlook" id="tourButton"><span class="buttontext">Outlook</span><i class="fas fa-chevron-right" id="arrow"></i><div id="tourrocket"></div></a></div><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Lee, H., Wiegand, D. J., Griswold, K., Punthambaker, S., Chun, H., Kohman, R. E., &amp; Church, G. M. (2020).</p><cite>Photon-directed multiplexed enzymatic DNA synthesis for molecular digital data storage.</cite><p><span class="journalTitle">Nature Communications</span> <span class="journalInfo">11(1).</span></p><a class="in-text" href="https://doi.org/10.1038/s41467-020-18681-5" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Photon-directed multiplexed enzymatic DNA synthesis for molecular digital data storage." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Green, N. M. (1975).</p><cite>Avidin.</cite><p><span class="journalTitle">Advances in Protein Chemistry.</span> <span class="journalInfo">29, 85–133.</span></p><a class="in-text" href="https://scholar.google.com/scholar?q=Avidin." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li></ol></article></div></div></div></nav><footer><div class="container"><div class="us"><h1>iGEM Aachen 2021</h1><h3>Follow Us</h3><div class="socialmedia"><a href="https://www.instagram.com/igem.Aachen/"><i class="fab fa-instagram-square fa-2x"></i></a><a href="https://www.facebook.com/iGEMAachen"><i class="fab fa-facebook-square fa-2x"></i></a><a href="https://www.youtube.com/channel/UCnIsgLJVUFhwqUYJtdd2tIA/videos"><i class="fab fa-youtube-square fa-2x"></i></a></div><h3>Contact Us</h3><div class="socialmedia"><a href="http://igem.rwth-aachen.de"><i class="fas fa-globe fa-2x"></i></a><a href="mailto:iGEM@rwth-aachen.de"><i class="fas fa-envelope-square fa-2x"></i></a></div></div></div><div class="sponsors"><div class="container"><img src="../assets/img/all_sponsors.png"></div></div></footer><script src="../content.bundle.js"></script></body></html>